
%%% This file contains the Background section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Background}\label{background}











\subsection{Gene expression}\label{gene-expression}

\textbf{MUUTA TÄTÄ KAPPALETTA!} Molecular biology is concerned with the study
of different molecules and their interactions at the cellular level and the
effects these have on cellular processes. Central species to molecular biology
are deoxyribonucleic acid (DNA), ribonucleic acid (RNA) and proteins; these
are the main actors of different cellular functions. The Central Dogma of
Molecular Biology, postulated by Crick first in 1958 and later in 1970 and
illustrated in figure \ref{central-dogma}, describes the general schema of
how genetic information flows from genes to proteins; genes are first
transcribed into messenger RNA (mRNA), which is then translated into protein
\citep{Crick1970,T0}.

The genome refers to the whole genetic material of a species or an individual.
The human genome has been suggested to contain approximately 20500 protein-
coding genes, which encompass only around 1.5\% of the whole genetic sequence
\citep{Clamp2007,MistäSilvaKeksinyTonProssan}. Recently, the ENCODE project...




\subsubsection{Regulation of gene expression}\label{regulation-of-gene-expression}





\subsubsection{Ribonucleic acids}\label{ribonucleic-acids}







\subsubsection{MicroRNAs}\label{micrornas}

MicroRNAs (miRNAs) are a class of noncoding, small RNA molecules that function
as post-transcriptional regulators of gene expression. This section serves as
an introduction to microRNAs, their biogenes, function, and regulation.

The first microRNA, lin-4, was discovered in 1993 by two research groups
studying the larval development of the nematode \emph{Caenorhabditis elegans}.
The groups noted that lin-4 does not encode a protein, but instead produces a
pair of small RNAs \citep{Lee1993,T1,T2}. The RNAs encoded by lin-4 have several
conserved antisense complementarity sites in an untranslated region of the
lin-14 mRNA, and these sites are necessary and sufficient for the normal
downregulation of lin-14 expression by lin-4 \citep{Lee1993,Wightman1993}. It
should be noted, that one of these groups also showed that lin-4 reduces the
amount of the LIN-14 protein -- the end-product of the lin-14 gene -- without
significantly affecting the cellular concentration of the lin-14 mRNA
\citep{??}.
%We will return to the issue of microRNA action in chapter \ref{microrna-function}.

Since then,
\textbf{microRNAs have been found in more than 200 organisms, including plants
\citep{CITE}. MiRNAs have been found to regulate and control several key cellular
processes, including LISTAAA \citep{CITE}.} Many miRNAs are also highly
conserved in evolution \citep{Bartel2004}, highlighting the importance of
their regulatory function.

In their mature form these tiny regulatory RNAs are single stranded and
approximately 20 to 23 nucleotides long. MiRNAs are not translated into
protein -- hence noncoding -- rather, they regulate gene expression by
influencing the translation of messenger RNAs (mRNAs). They assert
extensive control over the transcriptome; it has been estimated that more
than 60 \% of human mRNA transcripts are regulated by miRNAs
\citep{CITE}.

MiRBase is the de facto reference database of known microRNAs. It was
originally released in 2002, as the miRNA Registry, to serve as a
comprehensive konwledgebase of published microRNAs and also as an independent
authority on microRNA naming \citep{Griffiths2004}. Its latest version,
miRBase 21, was released in June 2014 \citep{MiRBaseWeb} and provides sequence
data, annotations as well as predicted and validated target genes for miRNAs
\citep{Kozomara2014}. The number of records in the database has risen
exponentially from 218 precursor and 218 mature miRNAs in the first release in
2002 to 28645 precursor 35828 mature miRNAs in 223 species in the current
version \citep{VanPeer2014,MiRBaseWeb}, illustrating the vast amount of novel
microRNA molecules discovered recently, mainly due to increasing efforts in
sequencing. MiRBaseTracker provides a service to update miRNA names and
annotations across different versions of miRBase to allow correct comparison
of miRNA study results and reannotation of miRNA analysis platforms
\citep{VanPeer2014}.





\paragraph{MicroRNA biogenesis}\label{microrna-biogenesis}

The biogenesis of microRNAs is illustrated in figure
\ref{fig:mirna-biogenesis} and is presented here as reviewed by  
\citet{Bartel2004} and \citet{Denzler2015}. MicroRNAs are
transcribed by RNA polymerase II. miRNAs are arranged in the genome as
single miRNA genes, with their own promotor regions,
or in polycistronic miRNA clusters that share a common promotor and are
transcribed simultaneously, or in same cases within introns or even exons of
other genes, which can be protein coding or noncoding.

The transcribed miRNA molecule is called a pri-miRNA. The pri-miRNA is cut by
the endonuclease Drosha (also known as Rnasen) to form one or several pre-
miRNAs. In the case of miRNAs located within introns of other genes
(mirtrons), the pre-miRNA is formed in RNA splicing by the spliceosome. Pre-
miRNA contain two inverted repeats, which fold together to form a
characteristic hairpin RNA molecule. A typical pre-miRNA structure is shown in
figure \ref{fig:premirna-structure}. The hairpin is then exported from the
nucleus to the cytoplasm.

In the cytoplasm, the endonuclease Dicer cleaves the head of the hairpin to
form a double-stranded miRNA/miRNA* duplex. The duplex is then bound by a
protein complex containing an argonaut protein, in humans one of Ago1-4. The
argonaut separates the strands of the duplex and retains one of them. The
retained strand is known as the guide strand (or miRNA). The other strand,
called the passenger (miRNA*), is released and degraded.

There exist exceptions to this general scheme. Some miRNAs are not dependent
on Drosha, such as mirtrons, which are cut into pre-miRNA by the spliceosome.
Other miRNAs are independent of Dicer, including miR-451 WHICH JOTAIN.
Additionally, in some cases the miRNA* can also be bound to an miRISC and used
as a miRNA \citep{CITE}.





\paragraph{MicroRNA function}\label{microrna-function}

The microRNA bound to an argonaut form the core part of a protein complex
known as the microRNA-induced silencing complex (miRISC). The miRISC seeks
messenger RNA molecules that match the bound template miRNA and binds them.

\begin{itemize}
\item
  RISC ja Argonaut
\item
  base pairing of seed sequence - sequence similarity families!
\item
  target degradation or inhibition of translation
\item
  combinatorial effect (many-to-one)
\end{itemize}





\paragraph{MicroRNA regulation and dysregulation}\label{microrna-
regulation-and-dysregulation}

MicroRNAs themselves are regulated by several different mechanisms, similarly
to mRNA and gene expression. Possible control points include transcription of
genomic miRNAs, post-transcriptional modification of pri- and pre-miRNAs,
JATKA LISTAA. MiRNAs also form transcriptional feedback loops with proteins
that act as transcription factors (TFs) for miRNAs. For example, JOKU TÄHÄN JA
KUVA SIITÄ. Some of these regulation mechanisms have a known important
biological function, such as miR-XXX and miR-XXX which function as a cell fate
switch, deciding the differentiation of ASF.

Dysregulation of microRNAs can happen at any of these putative regulation
steps, and many, if not all, of them have been implicated in tumorgenesis.
Examples of this include LISTAA. There are also additional mechanisms that can
lead to miRNA dysregulation, such as mutations in the miRNA sequence or the
target mRNA sequence, genomic rearrangements,
\textbf{LISÄÄ?}. The role of miRNAs in cancer has recently been
thoroughly reviewed by FOO-ET-AL \citep{CITE}.

It has also been shown that inhibiting the miRNA biogenesis pathway leads to
death in several organisms \citep{CITE} and, further, that perturbing the
biogenesis often leads to tumorgenesis \citep{CITE}. This highlights the
importance of microRNAs and their regulatory function.












\subsection{Cancer}\label{cancer}

Cancer is a disease of uncontrolled overgrowth of a population of cells. It is
generally viewed as a genetic disease, albeit mostly not inherited, as it is
caused by  mutations in the tumor genome. These mutations cause malfunction
and dysregulation of the genetic machinery regulating cellular functions, such
as cell proliferation, differentiation, and apoptosis, resulting in
unregulated growth and malignant tumor formation.

There are several classes of genes that promote tumor growth, the two main
categories being oncogenes and tumor suppressors. Oncogenes were first
identified in retroviruses and later shown to be proto-oncogenes, which by
mutation develop into oncogenes whose over activity promotes tumor growth
\citep{Varmus1988}. Tumor suppressor genes are often regulators of cell
proliferation or other so called housekeeping genes that work to ensure the
proper functioning of cells and the apoptosis of misbehaving ones. The
inactivation of these genes then leads to tumor progression. The existence of
tumor suppressors was first hypothesis by Alfred Knudson, who formed the “two-
hit hypothesis” while studying the epidemiology of retinoblastoma
\citep{Knudson1971}. He suggested that, for cancer to develop, both copies of
a tumor suppressor gene should become inactivated and that in inherited
cancers one mutation is acquired in the germline and the other occurs in
somatic cells, whereas in sporadic cancers both mutations happen in somatic
cells.

The idea of oncogenes and tumor suppressor genes was later expanded on by
Douglas Hanahan and Robert Weinberg in their seminal article The Hallmarks of
Cancer \citep{Hanahan2000}. The hallmarks are a set of six features which
tumors often acquire to become cancerous. The features are: sustaining
proliferative signaling, evading growth suppressors, resisting cell death,
enabling replicative immortality, inducing angiogenesis, and activating
invasion and metastasis. Weinberg and Hanahan postulated that at least three
of these six features are required for invasive cancer to develop.

Recently, Hanahan and Weinberg updated the hallmarks with two new upcoming
hallmarks and two enabling characteristics \citep{Hanahan2011}. The new
hallmarks are deregulation of cellular energetics and avoiding immune
destruction. The enabling characteristics of cancerous tumors are genome
instability and mutation, and tumor-promoting inflammation through recrution
of the immune system. Genome instability and mutation is of special importance
as much of cancer and tumor research focuses  on identifying mutated or
aberrantly expressed genes that promote tumor progression.



\subsubsection{Breast cancer}\label{breast-cancer}

Breast cancer constitutes a significant health issue globally. It is the most
common cancer in women and the second most common cancer overall.
Approximately 1.7 million women develop breast cancer annually world-wide, and
in 2012 there were 522 000 breast cancer related deaths.
\citep{Ferlay2015} In Finland there are approximately 4 800 new cases of
breast cancer annually, and there were 874 breast cancer related deaths in
2013 \citep{Syoparekisteri}.

Most breast cancers are sporadic; only 5-7\% of breast cancer cases are of
familial type \citep{Melchor2013}. However, 15-30\% of breast cancer patients
have a family member or relative with breast cancer. This is due to the high
frequency of breast cancer in many western populations, but also suggests that
there are unknown genetic factors and environmental factors that have an
impact on breast cancer development.  Indeed, breast cancer is a hormone
related disease and hormonal factors are known to have a great impact on
breast cancer risk.


\paragraph{Breast cancer classification}\label{breast-cancer-classification}

\begin{itemize}
\item
  histological types, (NMT classification?)
\item
  PAM50 molecular classification
\item
  new three-gene-model classification \textbf{tän edut? onko kukaan viitannu/käyttäny tätä?}
\end{itemize}




\subsubsection{MicroRNAs in breast cancer}








\subsection{Measurement of gene expression}\label{measurement-of-gene-expression}

\begin{itemize}
  \item blotting, q-PCR
  \item microarrays
  \begin{itemize}
    \item two-color arrays
    \item oligonucleotide arrays
    \item miRNA arrays
  \end{itemize}
  \item sequencing-based methods
  \item microarray analysis
  \begin{itemize}
    \item preprocessing and normalization
    \item probe annotation problems
  \end{itemize}
  \item protein arrays (RPPA)
\end{itemize}

\subsubsection{Microarrays}







\subsection{MiRNA target prediction}\label{mirna-target-prediction}

This section presents computational methods that have been used to predict
putative target genes for microRNAs and regulatory networks between genes and
microRNAs.





\subsubsection{Sequence-based target prediction}\label{sequence-based-target-
prediction}

\textbf{SELITÄ TÄÄ HOMMA JA ERI ALGORITMIT.}

Sequence-based prediction suffers from two major drawbacks. First, there are
high numbers of false positives \textbf{SYY JA VIITE
\citep{Sethupathy2006?}}. Second, the predictions are static and do not
account for different tissues or disease states.


\paragraph{MicroRNA target databases}





\subsubsection{Integration of expression data with sequence data}\label
{integration-of-expression-data-with-sequence-data}

Recently, a plethora of new methdos have been published that integrate
sequence-based target prediction with expression data. This helps combat the
high false-positive rate of sequence-only methods and enables tissue and
disease specific support for the predictions in real-world data. Recent
evidence indicates that miRNAs act predominantly through degradation
\citep{CITE}. Thus, it is feasible to use miRNA and mRNA expression data to
infer target relationships, since the regulatory effect of miRNAs should be
directly reflected in mRNA levels. All proposed methods use sequence-based
predictions as a starting point by considering only miRNA-mRNA pairs predicted
by at least one sequence-based prediction algorithm. This section provides a
review of algorithms for integrating miRNA and mRNA expression and
implementations in select target prediction methods.

\textbf{KOLME KATEGORIAA: KORRELAATIO, REGRESSIO, BAYESIAN?}



\paragraph{Correlation methods (incl MI)}\label{correlation-methods}

Mutual information (MI) is a simple measure of similarity between two
variables. \textbf{SELITÄ MI TARKEMMIN JA KAAVAN KANSSA JA LÄHDE} Thus, MI can
be used to measure the interdependence of miRNA-mRNA pairs from expression
data. However, MI does not distinguish the direction of the interaction, which
is highly relevant for miRNAs that are believed to mostly downregulate mRNA
expression. This constitutes a major drawback.

Correlation \textbf{SELITÄ KORRELAATIO JA LÄHDE}.

MAGIA \citep{Sales2010} is a webservice that implements both the MI and
correlation approaches. It also constructs a bipartite network of the top 250
predicted miRNA-mRNA pairs and provides links to several databases for further
examination and validation of results.

\textbf{VAI LAITTAISKO MAGIAN JO MI:N YHTEYTEEN JA TOISTAA VAAN
KORRELAATIOSSA?}



\paragraph{Regression methods}\label{regression-methods}

\textbf{SELITÄ REGRESSIO.}

Engelmann et al used least angle regression to show that gene expression can
be predicted from miRNA expression \citep{Engelmann}.

\textbf{SELITÄ REGULARISOIDUT/SHRINKAGE-TYYPPISET REGRESSIOMALLIT.}

While aiding with interpretability, shrinkage also has several drawbacks.
First, only a limited number of covariates may be included in the model, and
thus some relevant associations can be missed by number of included covariates
alone \citep{vanIterson2013}. Second, shrinkage may remove covariates highly
associated with and functionally regulating the response, instead retaining an
uninvolved covariate that correlates with actual regulators \citep{Engelmann}.
Relating to both limitations, van Iterson et al showed for one dataset that
lasso did not consistently select highly correlated miRNA-mRNA pairs
\citep{vanIterson2013}.

\textbf{SAISKO NÄIHIN VIITTEET REGRESSIOTEORIASTA EIKÄ TOSTA PAPERISTA?}


\subparagraph{Global test (regression)}\label{global-test-regression}

van Iterson et al recently proposed a miRNA target prediction method based on
the global test \citep{vanIterson2013}. \textbf{SELITÄ GLOBAL TEST}.

The method by van Iterson et al uses TargetScan, microCosm and PITA for
putative sequence-based targets and is available as an R package called
miRNAmRNA \citep{vanItersonWeb}.





\subsubsection{Bayesian methods}\label{bayesian-methods}

First published Bayesian analysis by Thomas Bayes in 1763 \citep{Gelman2013}.

Advantages of Bayesian methods:
\begin{itemize}
  \item
  quantification of uncertainty as probabilities, knowledge about anything
  unknown is described as a probability distribution, easy to comprehend
  \item
  common-sense interpretation of credible intervals compared to frequentist
  confidence intervals
  \item
  flexibility allows constructing complex models with relative ease (e.g.
  hierarchical models)
  \item
  Adding more data sequentially is possible by using the previous posterior
  distribution as the new prior distribution. This can be especially useful in
  a clinical research context, where data are often collected within a long
  timespan.
\end{itemize}
The challenge in Bayesian analysis is setting up proper probability models for
the parameters and observations. \citep{Gelman2013}

The posterior distribution represents a compromise between the prior (and
hence, prior information) and the observed data with the data having a larger
effect as the sample size increases. \citep{Gelman2013}.


\paragraph{Simulation}\label{simulation}

Bayesian anaylis often involves simulation if the form of sampling from the
obtained posterior distribution. This is convenient -- and necessary -- when
the exact probability density function cannot be explicitly obtained through
integration. Additionally, simulation often has the advantage of pointing out
problems in the model specification when simulated values are extremely small
of large.

Simulation methods:
\begin{itemize}
  \item
  sampling from probability distributions (easy with modern pseudorandom
  number generators)
  \item
  \textbf{mikäseyksinkertasinonkaan}
  \item
  Gibbs
  \item
  Hamiltonian Monte Carlo (this is used by Stan)
\end{itemize}


\paragraph{Bayesian regression}\label{bayesian-regression}

\textbf{SELITÄ MITEN BAYES-REGRESSIO EROAA TAVALLISESTA.}
