%!TEX root = dippa.tex
%%% This file contains the Background section of my master's thesis.
%%% Author: Viljami Aittomäki








\section{Gene expression}\label{gene-expression}

Genetic information is encoded in molecules of deoxyribonucleic acid (DNA). A
gene is a section of DNA that serves as a template for a functional
ribonucleic acid (RNA) molecule. Gene expression refers to this process of
synthesizing a functional end-product from the information contained in gene.
DNA and gene expression serve as the basis of all currently known life.

Most of gene expression is dedicated to making proteins. The Central Dogma of
Molecular Biology, postulated by Francis Crick in 1970, describes the general
schema of how genetic information flows from genes to proteins; DNA is first
transcribed into messenger RNA (mRNA), which is then translated into a polypeptides,
which ultimately form proteins \citep{Crick1970}. The flow is not strictly
one-directional though, as there exist reverse transcriptases, enzymes that
synthesize DNA from an RNA template.

All genes are read to form RNA, but not all encode a protein product. The
human genome\footnote{The genome refers to the whole genetic material of an
organism or an individual.} has been suggested to contain approximately 20 500
protein-coding genes, which encompass only around 1.5\% of the whole genetic
sequence \citep{Clamp2007}. Therefore, the vast majority of the human genome
was previously thought to be without function and referred to as "junk DNA".
More recently, however, it has become increasingly evident, that noncoding
portions of the genome are often functional and possibly have important
cellular functions \citep{ENCODE}. Noncoding genes give rise to noncoding RNA,
a class of RNA molecules that are mostly involved in aiding and regulating the
expression of other genes. Different types of noncoding RNA and their
functions are summarized in Table \ref{table:rnas}.




\subsection{Regulation of gene expression}\label{regulation-of-gene-expression}

Regulation of gene expression refers to controlling the abundance of
the gene end-product a cell produces. This control is paramount so that cells
can respond to external signals, changes in their environment, damage and also 
\textbf{pass through} different developmental stages. Gene expression can be regulated
at any stage of the process and regulation can be divided into transcriptional
and post-transcriptional regulation.

Transcriptional regulation with e.g. TFs, methylation, adenylation, splicing.

Post-ranscriptional

MicroRNAs (miRNAs) and small interfering RNAs (siRNAs) are components of the
so called RNA interference (RNAi) pathway, which is a mechanism for regulating
gene expression post-transcriptionally. miRNAs and siRNAs are practically
interchangeable as substrates for RNAi, both act as target mRNA recognizing
templates, but have different biogenesis, where miRNAs are cut from endogenous hairpin
strucutres and siRNAs are processed from exogenous long double-stranded RNAs (dsRNAs).
\cite{Du2005} MicroRNAs are the focus of this study and are discussed in more
detail in the next section.

The Argonaute (Ago) family of proteins is closely associated with small RNAs and
Ago act as effectors in RNAi \cite{Ha2014}.

\begin{itemize}
  \item transcription factors
  \item post-transcriptional regulation
  \item translational factors
  \item protein degradation
\end{itemize}


\subsection{Measuring gene expression}\label{measurement-of-gene-expression}

The physical measurement of gene expression can be done on either the level of
messenger RNA molecules or protein molecules present in cells. Although
proteins are the eventual effectors molecules within cells -- atleast for
protein-coding genes -- often gene expression is synonymous with mRNA
expression. This is because measuring mRNA abundances is significantly easier than
measuring protein abundances due to the chemistry of base-pair formation
and the relative ease of replicating DNA (or RNA) sequences by
exploiting cellular machinery evolved for this purpose.

Techniques shortly in one paragraph:
\begin{itemize}
  \item blotting, qPCR
  \item microarrays
  \item sequencing-based methods
  \item protein arrays (RPPA)
\end{itemize}


\subsubsection{Microarrays}

\begin{itemize}
  \item basics of operation
  \item shortly on microarray analysis
  \begin{itemize}
    \item preprocessing and normalization
    \item probe annotation problems
  \end{itemize}
\end{itemize}


The general assumption has been that mRNA expression is representative of gene
expression and that changes in mRNA abundances also reflect changes in protein
abundances and, therefore, cellular processes. This assumption has recently
been challenged by experiments indicating that the expression of mRNA and
correspoding protein often correlate poorly, with mRNA expression 
explaining around 40\% of variation in protein abundances \citep{Vogel2012}.
There are also contrary findings of modest to good correlation
and one study suggested that mRNA-protein correlation is generally higher for
genes that have differing mRNA expression between studied conditions
(e.g. cancerous versus healthy tissue) \citep{Koussounadis2015}.

Nonetheless, Payne recently concluded that "proteome and transcriptome abundances are not
sufficiently correlated to act as proxies for each other" and that
most of this difference is likely caused by biological regulation and not
by measurement technology \cite{Payne2015}.
% This regulation can be post-transcriptional, translational
% or protein-degradation related, as discussed above.
Therefore, it is interesting, even necessary, to integrate
measurements from different stages of gene expression -- for example
mRNA, microRNA and protein abundances -- to gain new insights
into biological processes.




















\section{MicroRNAs}\label{micrornas}

MicroRNAs (miRNAs) are a family of endogenous (i.e. coming from within the
cell itself) noncoding small RNA molecules that function as post-transcriptional
regulators of gene expression \citep{Ambros2004}. In their
functional, mature form miRNAs are single stranded and approximately 22
nucleotides long. MicroRNAs are not translated into protein. Instead, they
have an important role in regulation of gene expression in a wide range of
physiological, developmental and pathological processes \citep{Bartel2009}.
MicroRNAs assert their regulatory function by destabilization and degradation
of target messenger RNA (mRNA) molecules and inhibition of mRNA translation
\citep{Fabian2010}.




\subsection{Discovery of microRNAs}

The first known microRNA, lin-4, was discovered in 1993 by two research groups
studying the larval development of the nematode \emph{Caenorhabditis elegans}.
The researchers noted that lin-4 does not encode a protein, but instead
produces a pair of small RNAs, the longer of which was proposed to be a
precursor to the shorter one \citep{Lee1993}. The RNAs encoded by lin-4 were
noted to have conserved antisense complementarity in several sites of an
untranslated region of the lin-14 mRNA, and these sites were found to be
necessary for the normal repression of lin-14 expression by lin-4
\citep{Lee1993,Wightman1993}.
%It
%should be noted, that one of these groups also showed that lin-4 reduces the
%amount of the LIN-14 protein -- the end-product of the lin-14 gene -- without
%significantly affecting the cellular concentration of the lin-14 mRNA
%\citep{??}. %oks tää Wightman1993:ssa?
%We will return to the issue of microRNA action in chapter \ref{microrna-function}.

Let-7, the second microRNA to be discovered, was also first found in \emph{C.
elegans}, however, homologues of let-7 were found in several other species
\citep{Pasquinelli2000}. Soon after, numerous microRNA genes were found across
a variety of species, and a registry was set up to serve as a comprehensive
knowledgebase of published microRNAs and as an independent authority on
microRNA nomenclature \citep{GriffithsJones2004}. This registry later became
miRBase, the de facto reference database of known microRNAs, and now provides
sequence data, annotations as well as predicted and validated target genes for
miRNAs \citep{Kozomara2014}.

The number of known small RNAs has since vastly expanded
and microRNAs have been found in more than 200 organisms, including
all studied animals, plants \citep{JonesRhoades2006} and viruses \citep{Grundhoff2011}. 
The number of records in miRBase has risen exponentially
from %218 precursor and
218 mature miRNAs in the first release in 2002 to %28645 precursor and
35828 mature miRNAs in 223 species in the most recent version (v21, released June
2014 \citep{VanPeer2014,MiRBaseWeb}), illustrating the vast amount of novel
microRNA molecules discovered recently, mainly due to increasing efforts in
and availability of sequencing. miRBase lists 2588 known human miRNAs at the
time of writing this thesis. A web service called miRBaseTracker has been
developed by \citet{VanPeer2014} for updating miRNA nomenclature and
annotations across different versions of miRBase to allow correct comparison
of miRNA study results and reannotation of miRNA analysis platforms.




\subsection{MicroRNA genomics}\label{microrna-genomics}

MicroRNAs are highly conserved in evolution \citep{Bartel2004}, for example,
55\% of \emph{C. elegans} miRNAs have homologues in humans
\citep{IbanezVentoso2008}. Interestingly, the
appearance of multi-cellular organisms appears to correlate with the
appearance of the microRNA machinery, and organism complexity and speciation
are correlated with miRNA complexity, suggesting that microRNAs have had a
crucial role in the development of complex organisms \citep{Lee2007}.

MicroRNAs are found in varying genomic contexts. Approximately 50\% of
mammalian miRNAs are located in close proximity to other miRNAs and form
polycistronic miRNA clusters that are transcribed simultaneously. Some miRNAs
reside in the genome as dedicated miRNA genes, with their own promotor regions.
\citep{Kim2009} MicroRNAs and miRNA clusters can be situated in exons or
introns of nonconding genes and some are found in introns of protein-coding genes
\citep{Du2005}. MicroRNAs located in introns are sometimes referred to as
mirtrons \citep{Ruby2007}.

MicroRNAs are expressed in all tissues, however, different tissues
have differing miRNA expression profiles \citep{Krol2010}. Many microRNAs also
have differing expression in different developmental stages
of some organisms, e.g. let-7 functions to control the transition
from the second larval stage to \textbf{WHAT} in \emph{C. elegans} \citep{Bartel2004}.

% Tight evolutionary control, extensive transcriptome
% targeting, and the fact that miRNAs and their associated proteins are one of
% the most abundant molecules in the cytoplasm \citep{Bartel2004} highlight the
% importance of microRNAs.




\subsection{MicroRNA biogenesis}\label{microrna-biogenesis}

The canonical pathway of microRNA biogenesis is illustrated in Figure
\ref{fig:mirna-biogenesis} and is presented here as reviewed by \citet{Bartel2004},
\citet{Melo2011}, \citet{Ha2014}, and many others. 

Most microRNAs are transcribed from genomic DNA by RNA polymerase II to form a
long primary microRNA (pri-miRNA) molecule \citep{Lee2004}. The pri-miRNA molecule
contains a hairpin structure, with a 33-bp double-helix stem and a terminal
loop, and flanking single-strand sequences, which are several hundreds or
thousands of nucleotides long \citep{Kim2005}. Some miRNAs within Alu repeat elements
can be transcribed by RNA polymerase III \citep{Borchert2006}.

The pri-miRNA is cut by the ribonuclease Drosha to form %one or, in the case of polycistronic miRNAs, several
a pre-microRNA (pre-miRNA), which consists of the hairpin and is
approximately 70 nt long \citep{Lee2003}. A typical pre-miRNA structure is
shown in Figure \ref{fig:premirna-structure}. Drosha is aided by the essential
cofactor DGRC8 (the protein product of a gene deleted in DiGeorge syndrome \citep{Shiohama2003})
and they form a complex known as the Microprocessor \citep{Gregory2004}.
The hairpin is
then exported from the nucleus to the cytoplasm by exportin 5 (XPO5), which is
a member of the nuclear transport receptor family \citep{Lund2004}.

In the cytoplasm, the ribonuclease Dicer cleaves out the loop of the hairpin
to form a 22-nt-long double-stranded miRNA:miRNA* duplex corresponding to
the stem of the hairpin \citep{Bernstein2001}.
Dicer associates with a cofactor, in humans TRBP (Tar RNA-binding protein),
which is not required for effective dicing of the pre-miRNA,
but acts to physically bridge the Dicer to an Argonaute protein
% for further miRNA processing
\citep{Chendrimada2005}.

The duplex is then bound by the Argonaute protein, in mammals one of Ago1
through Ago4, forming what is called the RNA-induced silencing complex (RISC).
The RISC is a protein complex containing Dicer, TRBP and Ago \citep{Gregory2005}.
Ago, aided by Dicer and TRBP, unwinds the strands of the duplex and retains one of
them. The retained strand is known as the guide strand (or miRNA). The other
strand, called the passenger (or miRNA*), is released and typically degraded \citep{Du2005}.
In some instances, either one of the strands can become the guide
or both can be used \citep{Czech2009}.
%Notably, the Dicer cleaving, duplex unwinding and eventual
%mRNA regulation activity are, in fact, all coupled and performed by RISC
%\citep{Gregory2005}.

Not all miRNAs are generated through this canonical pathway of microRNA
biogenesis. Some miRNAs are not dependent on Drosha, such as mirtrons, which
are cut into pre-miRNA by the spliceosome, a molecular complex responsible for
removing introns (and sometimes exons) from precursor mRNA \citep{Ruby2007}. The
biogenesis of miR-451 is independent of Dicer; miR-451, which
has an important role in erythropoiesis, is cleaved by Ago2 \citep{Cheloufi2010}.






\subsection{MicroRNA mechanism of action}\label{microrna-mechanism}

The RISC is the effector of RNA interference. Ago functions as the catalytic
engine of the RISC and the miRNA bound to it guides the RISC to target
messenger RNAs \citep{Filipowicz2008}. The microRNA mechanism of action is
illustrated in Figure \ref{fig:mirna-action}.

Target recognition is based on sequence complimentarity of the miRNA and mRNA.
In animal miRNAs this complimentarity is almost always limited \citep{Ambros2004}.
Nucleotides at positions 2-8 of the 5' end of the microRNA have been found
crucial to target mRNA matching; these nucleotides are termed the miRNA "seed sequence".
miRNA target sequences are mostly located in the 3' UTR (untranslated region)
of the mRNA transcript, but in some instances target sites also reside in the
coding region or 5' UTR of the mRNA \citep{Bartel2009}.
\textbf{TÄÄ KAPPALE PAREMMIN, MUITA FIITSUJA MUKAAN, KOSKA NIITÄ KÄYTETÄÄN PREDICTIOSSA, KTS GRIMSON.)}

The action of microRNAs is \textbf{related} through inhibition of mRNA
translation or destabilization and subsequent degradation of mRNA. The exact
mechanisms by which the miRNA and Ago induce translational repression or
destabilization of mRNA are unclear \citep{Filipowicz2008}. Translational
inhibition was earlier believed to be the major form of miRNA action in animals,
but recent evidence suggests that mRNA destabilization dominates \citep{Guo2010}.
In some cases the mRNA is directly cleaved by Ago. Ago2 is the only mammalian
Argonaute capable of cleavage, and this is assumed to require
extensive base-pair matching between the miRNA and mRNA \citep{Du2005}.
However, mRNA cleavage appears to be rare in animals (while much more common
in plants).

mRNAs bound to RISC accumulate in so called processing bodies (P-bodies),
which are known sites of mRNA catabolism and translational repression in the
cytoplasm. The localization in P-bodies, however, appears to be a consequence
of RNA silencing, not the cause, and is reversible  \citep{Eulalio2007}.

Interestingly, several alternative mechanisms of action for microRNA have been
reported. For example, some miRNAs can increase the translation of target mRNA instead of
repressing it \citep{Vasudevan2007}, miR-373
was found to target DNA promoter areas and act to induce gene transcription
\citep{Place2008}, and miR-328 targets a protein to prevent inhibition of mRNA
translation \citep{Eiring2010}. This illustrates the complexity and diversity of
microRNA biology and gene regulation in general.






\subsection{MicroRNA function}\label{microrna-function}

SIIRRÄ TÄMÄ AIEMMAKSI?

MicroRNAs have been found to participate in regulation of almost all studied
cellular processes, including embryo development, cell proliferation,
apoptosis and metabolism, by regulating protein expression \cite{}.

miRNAs assert extensive control over the transcriptome. More than 60\% of
human mRNA transcripts are predicted to be regulated by miRNAs and a single
transcript has target sites for several different miRNA families
\citep{Friedman2009}. Furthermore, a single microRNA can have as many as hundreds or
thousands of target mRNAs. The effect of a
single miRNA on its target tends to be subtle, usually causing less than a
2-fold change in protein expression \citep{Baek2008}. However, a single
miRNA can have multiple binding sites in one mRNA and multiple miRNAs
acting on the same target have additive, and in some cases even synergistic
effects \citep{Bartel2009}.

% , and depending also on 
% of multiple miRNAs acting in tandem can, however, be much more pronounced and
% even multiplicative, achieving changes in excess of 10-fold \citep{}.

% A recent review concluded that mRNA degradation is the
% predominant form of miRNA action in mammals \citep{Guo2010}.

It should be noted, however, that the functional role and importance of many
miRNA-mRNA interactions are unknown, even for validated interaction pairs.
Uncovering these roles is challenging because of the subtle regulatory effects
that miRNAs often have and, additionally, because of the complexity and
robustness of most cellular regulatory networks \citep{Bartel2009}.
Furthermore, experimentally validated target mRNAs exist only for a subset of
all known microRNAs. Nonetheless, discovering miRNA targets is a critical
step in understanding their function.

The dysregulation of microRNAs is associated with many human diseases
\citep{Jiang2009,VAIHDATÄMÄREFE}. The first example of such an association was, in
fact, found in cancer, when miR-15 and miR-16 were found to be suppressed in
chronic lymphocytic leukemia \citep{Musilova2015}. A disease-promoting role
for miRNAs has since been implicated in many different cancers, including
breast cancer \citep{Melo2011}.



\subsection{Measuring MicroRNA expression}

The same methods that have been employed for measuring mRNA (i.e. gene)
expression are generally applicable for measuring microRNA expression as well,
ranging from northern blotting to microarrays and more recent studies using next
generation sequencing (NGS) \citep{Huang2011}. However, as Hunt and colleagues
in a recent and thorough review point out, there are several challenges in
detecting miRNAs in particular \citep{Hunt2015}.

MicroRNAs are very short and only comprise approximately 0.01\% of RNA
typically extracted from a sample \citep{Dong2013}. This implies that miRNA
detection must be highly sensitive. MicroRNAs from the same family can differ
by only one base, which in turn requires high specifity to be able to
distinguish between family members. On the other hand, variation in miRNA
processing can result in slight sequence variations, or isoforms, of a single
miRNA, also known as isomiRs \citep{StaregaRoslan2011,Lee2010}. This means
high specifity or an incorrect reference sequence (e.g. that of a 
weakly-expressed isomiR) used for detection can cause inaccurate measurements.
IsomiRs may also have different functions resulting from altered target
specifity \citep{Chugh2012}. The existence of the pri-miRNA, pre-miRNA and
mature miRNA molecules provides an additional challenge for measurement
methods.
%, although differentiation between these maturation stages is not necessarily required.

Many of the challenges mentioned are solved by NGS, which is sensitive and
reliable in quantifying known microRNAs and identifying novel ones
\citep{Huang2011}. Sequencing can detect variations of even one nucleotide and
does not necessarily depend on previously identified sequences. However, not
all identified short RNAs are functional miRNAs, and NGS conveys its own set
of problems relating to high cost, significant computational complexity and
validation efforts to distinguish relevant data from noise
\citep{Chugh2012,Hunt2015}.

\textbf{TÄHÄN VIELÄ ERI PLATFORMIEN VERTAILUA JA PREPROSESSOINNISTA?}















\section{Cancer}\label{cancer}

Cancer is a disease of uncontrolled overgrowth of a population of cells. It is
generally viewed as a genetic disease, albeit it is mostly not inherited, as it is
caused by  mutations in the genome of the tumor. These mutations cause malfunction
and dysregulation of the genetic machinery regulating cellular functions, such
as cell proliferation, differentiation and apoptosis, resulting in
unregulated growth and malignant tumor formation.

There are several classes of genes that influence tumor growth, the two main
categories being oncogenes and tumor suppressors. Oncogenes were first
identified in retroviruses and later shown to be proto-oncogenes, which by
mutation develop into oncogenes whose over activity promotes tumor growth
\citep{Varmus1988}. Tumor suppressor genes are often regulators of cell
proliferation or other so called housekeeping genes that work to ensure the
proper functioning of cells and the apoptosis of misbehaving ones. The
inactivation of these genes can lead to tumor progression. The existence of
tumor suppressors was first hypothesized by Alfred Knudson, who formed the “two-
hit hypothesis” while studying the epidemiology of retinoblastoma
\citep{Knudson1971}. He suggested that, for cancer to develop, both copies of
a tumor suppressor gene should become inactivated and that in inherited
cancers one mutation is acquired in the germline and the other occurs in
somatic cells, whereas in sporadic cancers both mutations happen in somatic
cells.

The idea of oncogenes and tumor suppressor genes was later expanded on by
Douglas Hanahan and Robert Weinberg in their seminal article The Hallmarks of
Cancer \citep{Hanahan2000}. The hallmarks are a set of six features which
tumors often acquire to become malignant. The features are: sustaining
proliferative signaling, evading growth suppressors, resisting cell death,
enabling replicative immortality, inducing angiogenesis, and activating
invasion and metastasis. Weinberg and Hanahan postulated that at least three
of these six features are required for invasive cancer to develop.

Recently, Hanahan and Weinberg revised the hallmarks with two new upcoming
hallmarks and two enabling characteristics \citep{Hanahan2011}. The new
hallmarks are deregulation of cellular energetics and avoiding immune
destruction. The enabling characteristics of malignant tumors are genome
instability and mutation, and tumor-promoting inflammation through recruition
of the immune system. Genome instability and mutation is of special importance
as much of cancer and tumor research focuses on identifying mutated or
dysregulated genes that promote tumor progression.



\subsection{Breast cancer}\label{breast-cancer}

Breast cancer constitutes a significant health issue globally. It is the most
common cancer in women and the second most common cancer overall;
approximately 1.7 million women develop breast cancer annually world-wide, and
in 2012 there were 522 000 breast cancer-related deaths
\citep{Ferlay2015}. In Finland there were 5008 new cases of breast cancer
 and 815 breast cancer-related deaths in 2014 \citep{Syoparekisteri}.

Most breast cancers are sporadic; only 5-7\% of breast cancer cases are of
familial type \citep{Melchor2013}. However, 15-30\% of breast cancer patients
have a family member or relative with breast cancer. This is
mostly due to the high frequency of breast cancer in many western populations,
but also suggests that there are unknown genetic factors and environmental
factors that have an impact on breast cancer development. Indeed, breast
cancer is a hormone-related disease and hormonal factors are known to have an
impact on breast cancer risk. The most important of these are
estrogen and progesterone.

The most important heriditary forms of breast cancer are those related to a
tumor predisposition syndrome caused by mutations in the breast cancer 1
(BRCA1) and BRCA2 genes, which explain about 25\% familial breast cancer in
many populations. These, however are rare on the population level and familial
clusering of beast cancer is multifactorial and caused by moderate risk and
low risk genetic variations, which are much more common \citep{Melchor2013}.


\subsubsection{Breast cancer classification}\label{breast-cancer-classification}

TÄTÄ PITÄÄ TIIVISTÄÄ!

The basic classification of cancer is based on the site -- that is, the organ
or tissue, such as breast or colon, where the primary tumor develops. Cancers
occurring within each organ are, however, heterogeneous in their nature and
have different behavior and prognosis. Thus, tumors are secondarily classified
by morphology, the microscopic structure of the cancer tissue.

The morphological classification of breast cancer is based on the WHO
classification from 2003 and includes altogether 19 histological subtypes of
invasive breast cancer \citep{Tavassoli2003,Weigelt2009}. Of these invasive
ductal carcinoma not otherwise specified (IDC NOS), accounts for the by far
largest histological group. Additionally, the WHO classification of
breast cancer includes the
TNM classification; characterization of the primary tumor (T), lymph node
status (N) and distant metastasis (M) and stage grouping based on the TNM
data. cTNM (or TNM) refers to a clinical TNM classification -- based on imaging studies,
surgical exploration and similar studies -- and pTNM to a pathological
classification -- based on postsurgical pathological analysis of tissue
biopsies. All invasive cancers are also graded into well (I), moderately (II)
or poorly (III) differentiated tumors based on microscopical examination
% based on three features; tubule formation as an expression of glandular
% differentiation, nuclear pleomorphism and mitotic counts
with less differentiated tumors having worse prognosis \citep{Tavassoli2003}.

In the clinic many other tumor characteristics are used. These
include the age of patient, lympho-vascular invasion as well as expression of
estrogen (ER) and progesterone receptors (PR), and Her2, which are routinely
studied for breast cancers. Together these data can be used to group patients
into risk categories for prognosis and choice of treatment using for example
StGallen criteria \citep{Goldhirsch2007} or NIH criteria \citep{Eifel2001} or others.
The estrogen receptor also has a special role in treatment, as tumors that higly express ER
(termed ER-positive and covering 70\% of all cases) can be given antiestrogens
as endocrine therapy.

One of the problems with morphological classification of breast tumors is that
over 50\% do not show any particular features and are classified as IDC NOS,
in spite of the fact that tumors in this large group have very different
clinical outcomes. As mentioned above, tumors of the same tissue -- and even
of the same histological type -- are heterogeneous in nature and personalized
molecular diagnostics are required to exploit more targeted treatments. This
is even more critical in the case of de novo treatment resistance, where
tumors develop a molecular mechanism for resisting pharmacological therapies.
This is especially common with targeted therapies, for example ER-positive
tumors may acquire mutations to ER rendering antiestrogen therapy ineffective
\citep{Oesterreich2013}.

More recently, expression profiling has led to a suggestion
of new classification of breast cancers \citep{Perou2000,Sorlie2001}. By studying 65
samples from breast tumors with expression profiling Perou et al.
distinguished four subgroups based on gene expression, namely
ER+/luminal-like, basal-like, Erb-B2+ and normal breast \citep{Perou2000}.

Expression profiling has also led to development of new prognostic tests to
help to determine the need of adjuvant chemotherapy. These tests include
Oncotype DX (a 21-gene recurrence score), MammaPrint (a 70-gene test) and PAM50
(a 50-gene test), and have been evaluated by several groups and suggested to be
valid and promising, but their utility in clinical decision making remains
unclear \citep{Azim2013}. The PAM50 molecular classification ... 

\textbf{SUBTYPE KORRELAATIO PROGNOOSIIN JA HOITOON PITÄISI KERTOA PAREMMIN?}






\subsection{MicroRNAs and cancer}

Research has shown microRNAs to have important roles in tumor initiation,
progression and metastasis \citep{Lin2015}. MicroRNA expression signatures
correlate with numerous cancer features, such as tissue, staging, 
progression, prognosis and treatment response, and all studied cancers
have had miRNA expression profiles differing from healthy tissue, including breast
\citep{Calin2006}. In fact, microRNAs appear to be globally underexpressed in
cancers \citep{Lu2005}. Therefore, it seems clear that microRNAs participate
in many of the pathways resulting in the hallmarks of cancer.

Similarily to protein-coding genes, microRNAs can function as tumor
suppressors or oncogenes \citep{Lin2015}. In a meta-analysis of dysregulation
of miRNAs in breast cancer van Schoonveld found major oncogenic microRNAs in
breast cancer to include miR-10b, miR-21, miR- 155, miR-373, and miR-520c
\citep{vanSchooneveld2015}. They also list nine miRNAs as major tumor suppressors for breast cancer,
namely miR-125b, miR-205, miR-17-92, miR-206, miR-200, miR-146b, miR-126,
miR-335, and miR-31.

The genetic mechanisms for microRNA involvement in cancer are varied,
including mutations in miRNA or target mRNA sequence, chromosomal
rearrangements of the miRNA-encoding DNA regions and epigenetic changes in DNA
methylation or histones, leading to aberrant miRNA expression
\citep{Calin2006,Melo2011}. For example, a single-nucleotide polymorphism
in the microRNA miR-196a2 has been found to be associated with breast cancer
risk \citep{Gao2011}. A mutation in the sequence of estrogen receptor alpha,
in the target site of miR-453, has been suggested to be associated with a
lower breast cancer risk \citep{Tchatchou2009}, as an example of a mutation in
a target transcript affecting miRNA function. MicroRNA function can also be
altered by abnormalities in the miRNA-processing machinery, for example, a
mutation in the Dicer gene causes a tumor predisposition syndrome known as
DICER1 syndrome \citep{Slade2011}. Another example of this is apparent
dysregulation of Dicer and Drosha in breast cancer
\citep{Yan2012}.

The different subtypes of breast cancer, explained above, reflect the genetic
background of the tumor and, accordingly, the subtypes differ in their gene
expression profiles. This also applies for miRNA expression, the different
intrinsic subtypes have different miRNA expression profiles, suggesting their
importance in breast cancer evolution \citep{Blenkiron2007}. de Rinaldis et al
identified a 46-miRNA signature that could be used in differentiating the
intrinsic subtypes from each other \citep{deRinaldis2013}. In addition to
tumor development, many miRNAs have been found to modulate the response to
breast cancer therapies. These include chemotherapy, antiendocrine therapy,
radiotherapy and targeted therapies.

Accordingly, miRNAs have been studied as biomarkers for diagnosing cancer and
cancer prognosis \citep{}. Emmadi et al recently found let-7 expression to be negatively
correlated with the Oncotype DX recurrence score in breast cancer
\citep{Emmadi2015}. This corroborated with the earlier finding of let-7 being
downregulated in breast cancer stem cells (tumor cells possessing the ability
of self-renewal) \citep{Yu2007} and later research suggesting let-7 to act as
a tumor suppressor. Several miRNAs have also been associated with breast cancer
metastasis \citep{Chen2016}.

MicroRNAs also show promise as a novel therapeutic tool, several studies have
proposed miRNA-based cancer treatments in animal models, including for breast
cancer \citep{VanRooij2014}. However, more research this area is needed before
microRNA treatments are ready for the clinical setting.

% Cell-free biomarker:
% Lawrie C.H., Gal S., Dunlop H.M., Pushkaran B., Liggins A.P., Pulford K., Banham A.H., Pezzella F., Boultwood J., Wainscoat J.S., et al. Detection of elevated levels of tumour-associated microRNAs in serum of patients with diffuse large B-cell lymphoma. Br. J. Haematol. 2008;141:672-675.

% Van Rooij E., Kauppinen S. Development of microRNA therapeutics is coming of age. EMBO Mol. Med. 2014;6:851-864.
%  Garzon R., Marcucci G., Croce C.M. Targeting microRNAs in cancer: rationale, strategies and challenges. Nat. Rev. Drug Discov. 2010;9:775-789.



















\section{Computational identification of microRNA targets}

% This section presents computational methods that have been used to predict
% putative target genes for microRNAs and regulatory networks between genes and
% microRNAs.

% Considering the recent large body of research on microRNAs, and their
% potential utility as biomarkers and treatment targets, it is not surprising
% that a plethora of computational tools have been released to aid in microRNA
% research.

% A recent review covering many published methods for different tasks
% has been written by Akhtar et al \citep{Akhtar2016}.

Recognizing the targets of microRNAs is essential to understand their
biological function and role in disease such as cancer. Many target
interactions have been found in experimental laboratory studies. Common
methods for such studies include using cell lines and introducing exogenous
miRNAs by transfection or suppressing endogenous ones and measuring the
effect on mRNA or protein expression. For a dateiled review of experimental
methods, see for example Thomson et al \citep{Thomson2011}.

Several public databases list currently known experimentally validated
microRNA targets. Examples include DIANA-TarBase \citep{Vlachos2015} and
MirTarBase \citep{Chou2016}, which are both manually curated from published
literature, and MiRWalk \citep{Dweep2015}, which combines data from several
other databases using text mining.

Although recent advances in high-throuput methodologies, such as CLIP-seq,
have significantly increased the scale of experimental studies, experimental
identification of microRNA targets remains laborous and costly and many
methods rely on computational processing of results \citep{Vlachos2015}. To
this end, a wide range of computational tools have been developed to aid in
miRNA taget discovery. Computational approaches to target prediction
can generally be classified into solely sequence-based tools and
expression-data-based tools (which often include sequence-features).
This section presents an overview of published methods developed for the
task of target prediction.

% For more in-depth reviews, see for example \citep{Yue2009,Muniategui2013}.




\subsection{Sequence-based target prediction}

Sequence-based prediction methods focus on finding miRNA-mRNA pairs that
have complimentary sequences, basing on the fact that sequence complimentarity
is the primary determinant of miRNA targeting. Hence, they can
only determine pair-wise relationships. From a machine learning perspective,
pair-wise prediction of miRNA-mRNA target pairs is essentially a classification
problem, where the goal is to identify a set features (both of the miRNA and
mRNA) that allows classifying mRNAs as either a target or a non-target of any
given miRNA. Table \ref{table:sequence-methods} lists examples of sequence-
based methods.

\begin{table}
  \caption{Examples of sequence-based tools for miRNA target prediction.
  Ref. $=$ reference. Features used $=$ experimentally derived sequence
  features used by the method (see text for details). SVM $=$ support
  vector machine. HMM $=$ hidden Markov model}
  \label{table:sequence-methods}
  \centering
  \begingroup\small
  \begin{tabular}{ llllp{4cm} }
    \\[-1ex] \hline\hline
    \textbf{Name} & \textbf{Ref.} & \textbf{Type} & \textbf{Features used} & \textbf{Additional notes} \\
    \hline
    TargetScan  & \citep{Agarwal2015} & Rule based & i, ii, iii & The first published target prediction tool. \\
    TOINENRULE  & \citep{} & Rule based &  &  \\
    mirTarget   & \citep{} & SVM & i, ii, iii, iv, v &  \\
    rna22       & \citep{} & HMM and rule & ??? &  \\
    PicTar      & \citep{} & HMM &  &  \\
    TargetBoost & \citep{} & Genetic programming & none & The algorithm learns the used features from training data. \\
    \hline\hline
    \end{tabular}
    \endgroup
\end{table}

Most sequence-based approaches are essentially rule-based filters, where
features of both the miRNA and mRNA sequence are used to narrow down candidate
target lists \citep{Yeu2009}. These features are derived from earlier
experimental knowledge, and commonly used features include: (i) sequence
complimentarity between the seed region of the miRNA and 3' UTR of the mRNA
(see Section \ref{mirna-mechanism}), (ii) seed matches in the coding region or
5' UTR of the mRNA, (iii) evolutionary conservation of seed matches between
species, (iv) target site accessibility and (v) free energy of the bound
miRNA-mRNA duplex. These rule-based prediction methods are unsupervised, i.e.
no training data is used to form the classifier. Instead, the relevance of the
used features is decided by the method's authors.

% An example of a rule-based predictor is illustrated in Figure \ref{fig:rule-flow}.

Some methods have used a supervised approach, where a training data set
consisting of experimentally validated targets and non-targets -- often
obtained from expression data sets -- is used to train a classifier for
classifying miRNA-mRNA pairs. Most commonly a support vector machine (SVM) is
used as the classifier. The features used for classification are similar to
rule-based tools, namely sequence features, but supervised learning allows the
inclusion of many more features. For example mirTarget uses a set of 113
features including seed region matches, conservation and a range of different
sequence features from different parts of the miRNA sequence
\citep{mirTarget}. More complex approahces have also been applied,
such as genetic programming and using hidden Markov models (HMM) as a 
sequence generative model to estimate targeting probability.

% For a more thorough
% review of several different sequence features and algorithms using them,
% see the reviews by Yue et al \citep{Yeu2009} and Bartel \citep{Bartel2009}.

The advantage of sequence-based methods is that they are based on
experimentally derived knowledge on molecular mechanisms and represent causal
relationships. As such, the predictions are easy to interpret. Disadvantages
of sequence-based methods include: considering only pair-wise
interactions cannot capture combinatorial
effects; using sequence conservation misses poorly conserved species-specific
targets; requiring seed region matches cannot identify miRNA targets without
seed matches (while these appear rare, they should not be discounted
altogether \citep{Bartel2009}); a sequence match does not always confer %\citep{Grimson2007}
repression and could be functionally inactive; and, finally, rule-based
methods are static do not account for differing miRNA and mRNA expression
profiles in various tissues and disease states. There is also a general lack
of overlap between predictions from different sequence-based approaches,
suggesting that many results are spurious false positives \citep{Muniategui2013}.






\subsection{Expression-data-based target prediction}\label{expr-methods}

Integrating expression data with sequence-based target prediction helps combat
the high false-positive rate of sequence-only methods and, importantly,
enables tissue and disease specific support for target predictions in real-world
data. Recent evidence indicates that miRNAs act predominantly through
mRNA degradation \citep{Guo2010}. Thus, it is feasible to use mRNA or protein
expression data to infer target relationships, since the regulatory effect of
miRNAs should be reflected in mRNA and protein abundances. Sequence-based
predictions are often incorporated as a preliminary filter step to limit the
potential interactions examined. Examples of published methods using
expression data are listed in Table \ref{table:expression-methods}. Various
mathematical approaches, ranging from correlation to complex Bayesian models,
have been proposed for expression-based prediction. Notably, most methods
limit the studied relationship to repression by the miRNA. This has
been suggested to improve performance \citep{Muniategui2013}, but
has the limitation of not being able to detect positive regulation,
both direct and indirect (mediated through regulation of other mRNAs)
\citep{Engelmann2012}.

% Mathematical models used in such expression
% analyses range from simple similarity measures to regression and complex
% Bayesian models, and examples are covered in this and the next section.

% Expression-based prediction methods use mathematical models that range from correlation
% to complex Bayesian regression models. The idea is to find (negatively)
% correlating miRNA-mRNA pairs or to predict mRNA or protein expression
% from miRNA expression patterns using regression. In the case of multiple regression
% target prediction becomes essentially a variable selection, where 
% the goal is to select microRNAs that best predict mRNA expression and then
% classify the mRNA in question as a target of chosen miRNAs.

\begin{table}
  \caption{Examples of expression-data based approaches to miRNA target prediction.
  %See text for details on several methods.
  Ref. $=$ reference, Method $=$ statistical method used, Seq. data $=$ how the method utilizes sequence-based predictions.}
  \label{table:expression-methods}
  \centering
  \begingroup\small
  \begin{tabular}{ llllp{4cm} }
    \\[-1ex] \hline\hline
    \textbf{Name} & \textbf{Ref.} & \textbf{Method} & \textbf{Seq. data} & \textbf{Additional notes} \\
    \hline
    MAGIA     & \citep{Sales2010}      & Correlation, MI        & Filter & Also produces a bipartite network of miRNA-mRNA interactions. \\
    TaLasso   & \citep{Muniategui2013} & Lasso regression       & Filter &  \\
    Engelmann & \citep{Engelmann2012}  & Least-angle regression & None or filter & Least-angle regression is a specific implementation of lasso. \\
    miRNAmRNA & \citep{vanIterson2013} & Global test & Filter   & Uses mRNA expression profiles to predict miRNA expression. \\
    Joku GSE  &                        & Gene-set enrichment    & \\
    GenMir++  & \citep{Huang2007}      & Bayesian regression    & None or filter & \\
    \hline\hline
    \end{tabular}
    \endgroup
\end{table}

Let us henceforth define $y_k = [y_1k, \dotsc y_nK]$ as a vector of expression
values of mRNA $k (k = 1, 2, \dotsc, K)$ and $X_{n \times p} = [x_j] =
[x_{ij}]$ as the matrix of expression values of miRNAs $j (j = 1, 2, \ldots,
p)$ for observations $i (i = 1, \ldots, n)$. In the case of regression
analysis (discussed below), for simplicity of notation, the subscript of $y_k$
is suppressed, and $X$ denotes a matrix $X_{n \times p+1}$, where a constant
vector $x_0 = [1, \ldots, 1]^T$ has been added to account for the intercept
term.

\paragraph{Correlation}
Several methods and publications use a straightforward approach to identifying
miRNA targets by finding miRNA-mRNA pairs whose expression patterns are
similar across observations. This is achieved with simple measures of variable
association. Pearson correlation is widely used because of its simplicity
and intuitive interpretation. Pearson correlation between mRNA $k$ and miRNA
$j$ is defined as:
\begin{equation}
	\rho_{kj} = (y_k)_{\mu_k=0|\sigma_k=1}^T \cdot (x_j)_{\mu_j=0|\sigma_j=1},
	\label{eq:pearson}
\end{equation}
where $\mu_i=0|\sigma_i=1$ indicates normalization to zero mean and unit
variance. Significantly correlated miRNA-mRNA pairs are classified as putative
target interactions. Other measures used include Spearman correlation and
mutual information (MI). A crucial limitation of correlation analysis is being
restricticted to studying pair-wise associations. Single miRNAs often have a
small effect on mRNA expression, which leads to weak associations and,
therefore, low power to identify miRNA targets. This issue is worsened by a
large multiple-hypothesis testing problem when cosidering all possible miRNA-mRNA
pairs. Some approaches have used additional information, such as
differential expression analysis and sequence-based predictions, for limiting
examined miRNA-mRNA pairs to alleviate this to some extent
\citep{Muniategui2013}.
% The drawback of MI is that it only indicates similarity of the variables, but not
% the direction, or sign, of the relationship.

\paragraph{Multivariate linear regression}
Many proposed expression-based methods use some form of multivariate linear
regression (MLR) to examine the relationship between miRNAs and mRNAs.
Expression profiles of miRNAs are commonly used to predict the expression of a
single mRNA (or protein) (although the opposite has also been proposed).
Recently Engelmann and showed that miRNA expression can indeed be used to
predict mRNA expression \citep{Engelmann2012}. In the context of regression,
target prediction essentially becomes a variable selection problem, where the
goal is to choose a set of miRNAs that best predict mRNA expression (without
overfitting).

A linear regression model for mRNA expression $y$ is defined as
\begin{equation}
  \label{eq:linear-regression}
	y = \sum_{j=0}^{p} (\beta_j \cdot x_j + \epsilon_{j}) =  X \beta + \epsilon,
\end{equation}
where $\beta = [\beta_0, \ldots, \beta_p]$ is the vector of regression coefficients,
$\beta_0$ is the intercept term, $\epsilon$ is the error term, and $X$ is the
matrix of covariates, i.e. the miRNA expression vectors. The parameter of
interest is $\beta$, which determines to the contribution of each miRNA to the
response variable, i.e. mRNA expression. The regression error $\epsilon$
represents noise and fitting error caused by variation not captured by the
included covariates. $\epsilon$ is commonly assumed to be normally
distributed with equal variance and no correlation between observations,
giving the \emph{normal linear model}.
% Commonly the solution is obtained by
% least squares, which involves minimizing the objective function,
% \begin{equation}
% 	min\{ || y - X \cdot w ||_2 \},
% \end{equation}
% which is equal to the sum of squares of the residuals.

The advantage of using regression for target prediction is the ability to
model the effect of several miRNAs on one gene simultaneously.
% This is desirable as the effect of a single
% miRNA on mRNA expression can be small, as discussed above.
Simple MLR is not applicable in cases, where the number covariates is larger
than the number of observations (here $K > n$), because the linear model is
undetermined and a single solution cannot be obtained. This is quite common
in analysis of high-throughput biological data, for example microarray
expresssion data. Furthermore, simple MLR cannot solve the problem of
variable selection as the model is asymptotically
improved by adding more covariates, leading to overfitting.

\paragraph{Regularized regression}
Both the dimentionality problem and overfitting can be solved using regularized
regression. The most common approach is to apply regularized least squares,
where a penalty depending on the magnitude of the coefficients $w$
is applied to force them small. This entails minimizing the expression
\begin{equation}
	min\{ \left \| y - X \cdot w \right \|_2 + \lambda \cdot R(w) \} ,
\end{equation}
where the first term corresponds to fitting error (the sum of squared residuals),
$R(w_j)$ is the penalty function and $\lambda$ is a tuning
parameter that controls the amount of regularization. 
The 1-norm ($R(w) = \left \| w \right \|_1 = \sum_{j=0}^{j} |w_j|$)
is frequently used for regularization; this is referred to as lasso regression.
% , where 
% regularizations include the 1-norm ($R(w) = ||w||_1$) in
% lasso regression, the 2-norm ($R(w) = ||w||_2$) in ridge
% regression and a combination of these ($R(w) =
% \lambda_1||w||_1 + \lambda_2||w||_2$) in what is called
% elastic-net regression.
Lasso regression in effect forces the number of non-zero coefficients to be small,
leading to a sparse solution that chooses seemingly important covariates.
% where as ridge regression results in a solution
% where coefficients are small but mostly non-zero \citep{Muniategui2013}.
While regularization solves the dimensionality problem and improves
interpretability, it has several important drawbacks. First, regularization
may remove covariates highly associated with and functionally regulating the
response, instead retaining an uninmportant covariate that correlates with
actual regulators \citep{Engelmann2012}. Second, only a limited number of
covariates may be included in the model, and thus some relevant associations
can be missed by number of included covariates alone.% \citep{vanIterson2013}.
Relating to both limitations, van Iterson et al showed for one dataset that
lasso did not consistently select highly correlated miRNA-mRNA pairs
\citep{vanIterson2013}.

\paragraph{Other approaches}
Other suggested approaches used include the global test, which does jotain
\citep{vanIterson2013}, and approaches similar to gene-set enrichment
analysis, where the over-representation of sequence-predicted targets in
differential expression sets is considered indicative of a target
relationship. Le at al proposed an ensemble method, which
combines predictions from several separate algorithms to
build on the advantages and compensate for the drawbacks of each
\citep{Le2015}.
Several Bayesian approaches have also been proposed;
these are covered in the next section.


% A group of bioinformatics tools uses mRNA expression data to suggest
% potentially interesting miRNA-target interactions (MTIs).
% \begin{itemize}
%   \item
%   Input ist of interesting genes (e.g. DE vs normal)
%   \item
%   Look for miRNA regulation patterns in list (analogous to gene set enrichment REF REVIEW)
%   \item
%   Enriched miRNAs deemed interesting for this data
% \end{itemize}


% All proposed methods use sequence-based
% predictions as a starting point by considering only miRNA-mRNA pairs predicted
% by at least one sequence-based prediction algorithm. This section provides a
% review of algorithms for integrating miRNA and mRNA expression and
% implementations in select target prediction methods.

% Ainakin nämä vaikka:
% \begin{itemize}
%   \item
%   DIANA-mirExTra (uusin versio NGS-datalle)
%   \item
%   GeneSet2miRNA
%   \item
%   Sylamer(?)
% \end{itemize}

% These methods, however, rely on previous target site predictions based on
% miRNA and mRNA sequences, some of which may be outdated, and 
% cannot suggest novel target interactions. They also do not account for
% the differences in miRNA expression inherent to different tissues and
% possibly aberrant expression in diseases.


% \subsubsection{Integration of mRNA and miRNA expression}

% Since it is evident that miRNA regulation affects mRNA expression levels, as
% mentioned above, it seems feasible to predict MTIs by integrating miRNA and
% mRNA expression data. Many tools have been developed that do this using
% correlation, mutual information or various forms of regression.

% \textbf{KOLME KATEGORIAA: KORRELAATIO, REGRESSIO, BAYESIAN?}

% \paragraph{Correlation methods (incl MI)}\label{correlation-methods}

% Mutual information (MI) is a simple measure of similarity between two
% variables. \textbf{SELITÄ MI TARKEMMIN JA KAAVAN KANSSA JA LÄHDE} Thus, MI can
% be used to measure the interdependence of miRNA-mRNA pairs from expression
% data. However, MI does not distinguish the direction of the interaction, which
% is highly relevant for miRNAs that are believed to mostly downregulate mRNA
% expression. This constitutes a major drawback.

% Correlation \textbf{SELITÄ KORRELAATIO JA LÄHDE}.

% MAGIA \citep{Sales2010} is a webservice that implements both the MI and
% correlation approaches. It also constructs a bipartite network of the top 250
% predicted miRNA-mRNA pairs and provides links to several databases for further
% examination and validation of results.



% \paragraph{Regression methods}\label{regression-methods}

% \subparagraph{Global test (regression)}\label{global-test-regression}

% van Iterson et al recently proposed a miRNA target prediction method based on
% the global test \citep{vanIterson2013}. \textbf{SELITÄ GLOBAL TEST}.

% The method by van Iterson et al uses TargetScan, microCosm and PITA for
% putative sequence-based targets and is available as an R package called
% miRNAmRNA \citep{vanItersonWeb}.








\subsection{Bayesian approaches}\label{bayesian-approaches}

The foundation of Bayesian data analysis is quantifying uncertainty as
probability. Current knowledge about unknown parameters and variables is
described as probability distributions \citep{Gelman2013}.
This provides a framework which is inherently suited to dealing with noisy
real-world data, as measurement noise is naturally included in
probability statements.

\subsubsection{Basics of Bayesian analysis}

Bayesian analysis begins by defining a joint probability model $p(y,\theta)$
for observed data $y$ and unknown model parameters $\theta$.
The joint distribution can be written as a product of two probability distributions
\begin{equation}
  p(y,\theta) = p(\theta) p(y|\theta),
\end{equation}
which are referred to as the \emph{prior distribution} $p(\theta)$ and the
data distribution or \emph{likelihood} $p(y|\theta)$. The prior conveys
information on the presumed values a parameter may take and the likelihood
represents the likeliness of observed data for given parameter values (in the
context of the chosen data model). Applying the Bayes' theorem we obtain the
\emph{posterior distribution} for $\theta$ given the known values of the
observations $y$:
\begin{equation}
  \label{eq:bayes}
  p(\theta|y) = \frac{p(y,\theta)}{p(y)} = \frac{p(\theta) p(y|\theta)}{p(y)},
\end{equation}
where $p(y) = \int_{\theta} p(\theta) \cdot p(y|\theta) d\theta$.
Noting that $p(y)$ does not depend on $\theta$, we can write Equation
\ref{eq:bayes} as
\begin{equation}
  p(\theta|y) \propto p(\theta) p(y|\theta).
\end{equation}
The latter is referred to as the unnormalized posterior distribution. The
Bayes' theorem forms the heart of Bayesian inference and illustrates the core
concept of updating prior beliefs to account for observed evidence. The
posterior provides a probability assessment of the possible values of
a parameter, and represents a compromise between prior knowledge and information
contained in observed data. As the number of observations increases, the
data have increasing influence on the posterior \citep{Gelman2013}.

The virtue of modeling uncertainty as probabilities -- in addition to
naturally dealing with noise -- is that they are conceptually easy to grasp
and often allow common-sense interpretations of conclusions to be made.
\footnote{This is especially true compared to classical frequentist analysis,
which is defined within the context of repeated sampling (and inference) from
a fixed but unknown process generating the observations. For
example, frequentist confidence intervals strictly do not indicate that the
true value of the parameter is contained within with high probability -- a
common misconception -- where as Bayesian posterior intervals do (subject to
modeling assumptions, of course).}
Bayesian modeling is also flexible and can cope with complex problems with
relative ease. Prior and expert knowledge about the parameters of interest can be
embedded in the prior distribution and different priors may be assigned to
each parameter.
% additional data can be included sequentially using a
% previously obtained posterior as a new prior distribution.

Hierarchical models -- where parameters of the prior
distributions have their own priors called \emph{hyperpriors} -- provide
additional flexibility. In situations where model parameters are
related to each other, a common hyperprior may be used for several parameters.
Hierarchical models are particularly appropriate in settings where
data are sparse (as available information is be shared between parameters)
or the data are naturally structured into several levels, such as 
similar measurements from different hospitals or schools.

The challenge in Bayesian analysis is choosing proper probability models for
the parameters and observations, including prior distributions as well as the
likelihood \citep{Gelman2013}. In fact, Bayesian methodology has been
criticized for the subjectivity related to choosing suitable priors, as it is
based on the experience and reasoning of the statistician. One could, however,
argue that the choice of any model is always subjective to a certain degree,
irrespective of chosen methodology. Additionally, weakly informative or non-informative
priors can be used to decrease the effect of subjective information (or
in cases where no prior knowledge is available) and results from
inferences using non-informative priors often coincide with classical
analyses.


% In cases where no prior information is available, weakly informative
% or noninformative priors can be used. These convey little or no
% information on the presumed values a parameter may take, respectively.
% In many instances, using a non-informative prior results in similar or equal
% results as frequentist analysis, but the strength of Bayesian analysis comes
% from including prior knowledge in the prior distribution. \citep{Jaynes?} The
% posterior distribution represents a compromise between the prior (and, hence,
% prior information) and the observed data, with the data having an increasing
% effect as the sample size increases \citep{Gelman2013}. The posterior
% distribution also provides a more comprehensive view of
% one's knowledge on the parameter of interest than, say, a single confidence
% interval.

% The frequentist approach only considers the data to have a probability
% distribution, the likelihood. The process giving rise to the data, and the
% parameters that define it, are considered fixed. The observed data are
% assessed with respect to other data that might be generated by the same model.

% OMIN SANOIN: Confidence intervals work their best when you don't know much about a
% parameter beyond the information contained in a data set. And further,
% credibility intervals won't be able to improve much on confidence intervals
% unless there is prior information which the confidence interval can't take
% into account, or finding the sufficient and ancillary statistics is hard.



\subsubsection{Bayesian regression}

Bayesian regression analysis aims to infer the posterior distributions
for the regression coefficients of covariates and other model parameters,
such as the variance (i.e. the error term) of the observation model.
Within the Bayesian framework, the normal linear regression defined in
Equation \ref{eq:linear-regression} can be expressed as
\begin{equation}
  y | \beta, \sigma, X \sim N(X \beta, \sigma^2I),
\end{equation}
where $N$ is the multivariate normal distribution, and $I$ is the $n \times n$
identity matrix. The mean of $y$ is then the familiar linear sum of $x_k$
\begin{equation}
  \textrm{E}(y|\beta,X) = X \beta = \beta_0 + \beta_1 x_1 + \dotsb + \beta_p x_p.
\end{equation}
The posterior distribution for the regression coefficients (up to a
normalizing constant) is obtained from the marginal posterior
\begin{equation}
  p(\beta | \sigma, y, X) \propto \int p(y | \beta, \sigma, X) p(\beta | \sigma) p(\sigma) d\sigma,
\end{equation}
with the joint prior $p(\beta, \sigma) = p(\beta | \sigma) p(\sigma)$.
It is relatively straightforward to extend this simple model, for instance by
allowing unequal variances or correlation between observations, choosing a
different data distribution
% than the normal distribution
to represent the error term or including hyperparameters to construct a
hierarchical model.


\subsubsection{Variable selection in the Bayesian context}

As mentioned in Section \ref{expr-methods}, microRNA target prediction using
regression analysis of expression data is effectively a variable selection
problem. Many Bayesian variable selection methods have been published. A
detailed review of these falls outside the scope of this thesis, however, a
comprehensive one has been recently written by Piironen
and Vehtari \citep{Piironen}.

For Bayesian regression, several different priors allowing model
shrinkage have been proposed, including the Laplace prior
(which is closely related to lasso regression), the horseshoe prior,
and the hierarchical shrinkage prior. The latter has been shown
to generally outperform the others.

In this work a recently published Bayesian variable selection method called
projection prediction variable selection was used to infer miRNA-mRNA
target relationships. The details of this method
are presented in Section \ref{sec:PPVS}.



\subsubsection{Bayesian inference}\label{simulation}

The goal of Bayesian inference is often to make conclusions about uknown
parameters $\theta$ of uknown observations $\tilde{y}$ given observed data
$y$. These are formulated as probability statements.

Often it is not possible to obtain analytical solutions to integrals
. This is especially true for hierarchical models. It is therefore
oftentimes necessary to use numerical estimation or simulation to
derive estimates of the posterior.

% Bayesian analysis often involves simulation if the form of sampling from the
% obtained posterior distribution. This is convenient -- and necessary -- when
% the exact probability density function cannot be explicitly obtained through
% integration. Additionally, simulation often has the advantage of pointing out
% problems in the model specification when simulated values are extremely small
% of large.

% Often obtaining analytical solutions to integrals or explicit formulations of
% the posterior distribution is not possible. This is especially true for
% complex and hierarchical models. In these cases it is possible to use
% simulation to obtain samples from the posterior distribution of parameters and
% use these to compute estimates for parameters and other quantities of
% interest.

Simplest approaches to simulation include sampling directly from the posterior
distribution $p(\theta|y)$, where possible, or a distribution proportional
to $p(\theta|y)$. More commonly Markov chain methods are used. These entail
drawing iterative samples of $\theta$ from a distribution that approaches
the true posterior during iteration rounds, where each draw $\theta^t$ is
conditional (only) on the previous one $\theta^{t-1}$.
\footnote{This is essentially the definition of a Markov chain; a sequence of random
variables, where the probability density of each one is dependent on only the
previous one.} It is advisable to run several separate simulations with
different starting values $\theta^0$ to assure proper coverage of the whole
posterior. These are referred to as chains.

Markov chain simulations should always be assessed for proper converge, both
for each chain and between the chains. For this purpose, Gelman et al have
proposed the $\hat{R}$ measure\footnote{Not to be confused with $\bar{R}^2$,
the adjusted coefficient of determination of a regression model, defined in
the next section.}, which estimates the ratio of between-chain variation and
within-chain variation for each parameter \citep{Gelman2013}. Gelman et al
suggest $\hat{R}^2 < 1.1$ is a reasonable indicator of good convergence. It is
also good practice to discard the first simulation samples (commonly the first
half of each chain) to ensure that samples arise from a converged state.


\subsubsection{Previous Bayesian target-prediction methods}

% Bayesian analysis has previously been applied to miRNA target prediction.

One of the earliest tools to use expression data for target prediction was
GenMir++. It is based on A BAYESIAN SYSTEEMI, KIRJOITA AUKI.
The latest version of GenMir (GenMir3) now integrates sequence-based features
into the algorithm, although the authors note that this does not affect the
results to a significant extent.

Stingo et al have proposed a Bayesian method based on spike-and-slab variable
selection.

Previous Bayesian miRNA methods:
\begin{itemize}
  \item Stingo et al
  \item GenMir3/GenMir++
  \item Se joku graafinen (korkeintaan lauseen maininta)
\end{itemize}



