
%%% This file contains the Background section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Background}\label{background}











\subsection{Gene expression}\label{gene-expression}

\textbf{MUUTA TÄTÄ KAPPALETTA!} Molecular biology is concerned with the study
of different molecules and their interactions at the cellular level and the
effects these have on cellular processes. Central species to molecular biology
are deoxyribonucleic acid (DNA), ribonucleic acid (RNA) and proteins; these
are the main actors of different cellular functions. The Central Dogma of
Molecular Biology, postulated by Crick first in 1958 and later in 1970 and
illustrated in figure \ref{central-dogma}, describes the general schema of
how genetic information flows from genes to proteins; genes are first
transcribed into messenger RNA (mRNA), which is then translated into protein
\citep{Crick1970,T0}.

The genome refers to the whole genetic material of a species or an individual.
The human genome has been suggested to contain approximately 20500 protein-
coding genes, which encompass only around 1.5\% of the whole genetic sequence
\citep{Clamp2007,MistäSilvaKeksinyTonProssan}. Recently, the ENCODE project...
\textbf{Tämäkin on huono.}




\subsubsection{Regulation of gene expression}\label{regulation-of-gene-expression}





\subsubsection{Ribonucleic acids}\label{ribonucleic-acids}







\subsection{MicroRNAs}\label{micrornas}

MicroRNAs (miRNAs) are a family of noncoding, small RNA molecules that function
as post-transcriptional regulators of gene expression \citep{Ambros2004}. In
their functional, mature form miRNAs are single stranded and approximately 22
nucleotides long. MicroRNAs are not translated into protein -- hence
noncoding. However, they have an important role in regulation of gene
expression in a wide range of physiological, developmental and pathological
processes \citep{Bartel2009}. MicroRNAs assert their regulatory function by destabilization and
degradation of target messenger RNA (mRNA) molecules and inhibition of mRNA
translation \citep{Fabian2010}.



\subsubsection{History of microRNA discovery}

The first known microRNA, lin-4, was discovered in 1993 by two research groups
studying the larval development of the nematode \emph{Caenorhabditis elegans}.
The researchers noted that lin-4 does not encode a protein, but instead
produces a pair of small RNAs, the longer of which was proposed to be a
precursor to the shorter one \citep{Lee1993}. The RNAs encoded by lin-4 were
noted to have conserved antisense complementarity in several sites of an
untranslated region of the lin-14 mRNA, and these sites were found to be
necessary for the normal repression of lin-14 expression by lin-4
\citep{Lee1993,Wightman1993}.
%It
%should be noted, that one of these groups also showed that lin-4 reduces the
%amount of the LIN-14 protein -- the end-product of the lin-14 gene -- without
%significantly affecting the cellular concentration of the lin-14 mRNA
%\citep{??}. %oks tää Wightman1993:ssa?
%We will return to the issue of microRNA action in chapter \ref{microrna-function}.

The second microRNA to be discovered, let-7, was not found until seven years
later. Although let-7 was also first found in \emph{C. elegans}, homologues of
let-7 were then found in several other species, including human
\citep{Pasquinelli2000}. Soon after, numerous microRNA genes were found across
a variety of species, and a registry was set up to serve as a comprehensive
knowledgebase of published microRNAs and as an independent authority on
microRNA nomenclature \citep{GriffithsJones2004}. This registry later became
miRBase, the de facto reference database of known microRNAs, and now provides
sequence data, annotations as well as predicted and validated target genes for
miRNAs \citep{Kozomara2014}.

Since these early discoveries, the number of known small RNAs has expanded
remarkably and microRNAs have been found in more than 200 organisms, including
plants and viruses \citep{JonesRhoades2006,Grundhoff2011}. 
The number of records in miRBase has risen exponentially
from %218 precursor and
218 mature miRNAs in the first release in 2002 to %28645 precursor and
35828 mature miRNAs in 223 species in the current version (v21,
released June 2014 \citep{VanPeer2014,MiRBaseWeb}), illustrating the vast
amount of novel microRNA molecules discovered recently, mainly due to
increasing efforts in and availability of sequencing.
Currently, 2588 human miRNAs are listed in miRBase. A web service called
miRBaseTracker has been developed by \citet{VanPeer2014} for updating
miRNA nomenclature and annotations across different versions of miRBase to
allow correct comparison of miRNA study results and reannotation of miRNA
analysis platforms.



\subsubsection{MicroRNA genomics}\label{microrna-genomics}

Most miRNAs are located in the genome as dedicated miRNA genes, with their own
promotor regions, or in polycistronic miRNA clusters that are transcribed
simultaneously and share common expression patterns. However, some
microRNAs are situated in introns of protein coding genes \citep{Lin2015},
and others are found in exons or introns of nonconding genes \citep{Du2005}.
MicroRNAs located in introns are sometimes referred to as mirtrons \citep{Ruby2007}.
Examples of these are given in figure \ref{fig:mirna-locations}.

MicroRNAs are expressed in all tissues, however, different tissues
have different miRNA expression profiles \citep{Krol2010}. Many microRNAs also
have differing expression in different developmental stages
of some organisms, e.g. let-7 functions to control the transition
from the second larval stage to in \emph{C. elegans} \citep{Pasquelli2000}.

MicroRNAs are highly conserved in evolution \citep{Bartel2004}.
%, highlighting the importance of their regulatory function during evolution.
Accordingly, 55\% of \emph{C. elegans} miRNAs have homologues in humans
\citep{Ibanez-Ventoso2008}. Tight evolutionary control, extensive transcriptome
targeting, and the fact that miRNAs and their associated proteins are one of
the most abundant molecules in the cytoplasm \citep{Bartel2004} highlight the
importance of microRNAs.


\begin{itemize}
\item
  differences between animal and plant miRNAs (short mention)
\item
  evolutionary conservation (important!)
\end{itemize}


 



\subsubsection{MicroRNA biogenesis}\label{microrna-biogenesis}

The canonical pathway of microRNA biogenesis is illustrated in figure
\ref{fig:mirna-biogenesis}
%and is presented here as reviewed by \citet{Bartel2004}, \citet{Melo2015}, and several others. 

Most microRNAs are transcribed from genomic DNA by RNA polymerase II to form a
long primary microRNA (pri-miRNA) molecule \citep{Lee2004}. Some pri-miRNAs
have an alternative synthesis pathway, especially those associated with
genomic repeats. For example, miRNAs within Alu repeat elements are
transcribed by RNA polymerase III \citep{Borchert2006}. The pri-miRNA molecule
contains a hairpin structure, with a 33-bp double-helix stem and a terminal
loop, and flanking single-strand sequences, which are several hundreds or
thousands of nucleotides long \citep{Kim2005}.

The pri-miRNA is cut by the ribonuclease Drosha to form%one or, in the case of polycistronic miRNAs, several
a pre-microRNA (pre-miRNA), which consists of the hairpin structure and is
approximately 70 nt long \citep{Lee2003}. A typical pre-miRNA structure is
shown in figure \ref{fig:premirna-structure}. Drosha is aided by the essential
co-factor DGRC8 (the protein product of a gene deleted in DiGeorge syndrome)
and they form a complex known as the microprocessor \citep{Gregory2004}. In
the case of miRNAs located within introns, the pre-miRNA is formed during
precursor mRNA splicing by the spliceosome \citep{Ruby2007}. The hairpin is
then exported from the nucleus to the cytoplasm by Exportin-5 (XPO5), which is
a member of the nuclear transport receptor family
\citep{Lund2004}.
%The pre-miRNA still contains the two inverted repeats, which together form the now shortened hairpin RNA molecule.

In the cytoplasm, the ribonuclease Dicer cleaves out the loop of the hairpin
to form a 22-nt-long double-stranded miRNA/miRNA* duplex, which corresponds to
the stem of the hairpin \citep{Bernstein2001,Park2011}.
Dicer associates with a cofactor, in humans TRBP (Tar RNA-binding protein),
which is not required for effective dicing of the pre-miRNA \citep{Zhang2004},
but acts to recruit the Dicer complex to an argonaut protein
for further miRNA processing \citep{Chendrimada2005}.

The duplex is then bound by an argonaut protein, in mammals one of Ago1
through Ago4, in what is called the RNA-induced silencing complex (RISC), a
protein complex containing Dicer, TRBP and the Ago \citep{Gregory2005}. Ago,
aided by Dicer and TRBP, unwinds the strands of the duplex and retains one of
them. The retained strand is known as the guide strand (or miRNA). The other
strand, called the passenger (or miRNA*), is released and typically degraded .
In some instances, either one of the strands can become the guide strand
\citep{Ruby2006}.
%Notably, the Dicer cleaving, duplex unwinding and eventual
%mRNA regulation activity are, in fact, all coupled and performed by RISC
%\citep{Gregory2005}.

Not all miRNAs are generated through this canonical pathway of microRNA
biogenesis. Some miRNAs are not dependent on Drosha, such as mirtrons, which
are cut into pre-miRNA by the spliceosome, as mentioned above. The biogenesis of
miR-451 is independent of Dicer; miR-451 is cleaved by Ago2 and is
important for erythropoesis \citep{Cheloufi2010}. Furthermore, in some cases
the miRNA* is not merely degraded, but can also be used as a miRNA
\citep{Czech2009}.





\subsubsection{MicroRNA mechanism}\label{microrna-mechanism}

A microRNA bound to an argonaut form the core part of the miRISC (microRNA-
induced silencing complex) protein complex. The miRISC seeks messenger RNA
molecules that match the bound template miRNA and binds to them. This is illustrated
in figure \ref{fig:mirna-action}.

Nucleotides 2-7 of the mature miRNA sequence form the so-called seed sequence, which primarily
determines which mRNAs the miRNA will bind.

\begin{itemize}
\item
  RISC ja Argonaut
\item
  base pairing of seed sequence - sequence similarity families!
\item
  target degradation or inhibition of translation
\item
  combinatorial effect (many-to-one)
\end{itemize}





\subsubsection{MicroRNA function and role in disease}\label{microrna-function-and-role-in-disease}

MicroRNAs have been found to participate in regulation of a diverse range of
cellular processes, ranging from apoptosis and X to X and including
development, by regulation of protein expression \citep{Bushati2007}. miRNAs
assert extensive control over the transcriptome; it has been estimated that
more than 60\% of human mRNA transcripts are regulated by miRNAs
\citep{Friedman2009}.

Several studies have shown that miRNAs indeed affect the mRNA concentrations
of their targets within cells and, notably, a recent review concluded that mRNA
degradation is the predominant form of miRNA action in metazoans
\citep{Guo2010}. Additionally, many studies have shown that miRNAs also have an
effect on protein concentrations of the corresponding target mRNAs. \citep{AARGHMIKÄTÄÄVIITEON}

It should be noted, however, that the functional role of many miRNA-mRNA
interactions is unknown, even for validated interaction pairs. Uncovering
these roles is challenging because of the subtle regulatory effects that
miRNAs often have and, additionally, because of the complexity and robustness
of most cellular regulatory networks \citep{Bartel2009}. Furthermore,
validated target mRNAs exist only for a subset of all known microRNAs.

TÄHÄN KAPPALE SIITÄ, ETTÄ miRNAt MUKANA TAUDEISSA JA SYÖVÄSSÄ (EKA SYÖPÄ
JA MUITA).

Similarily to protein-coding genes, microRNAs can function as tumor
suppressors or oncogenes \citep{Lin2015}. The genetic mechanisms for microRNA
involvement in cancer can also be analogous, such as mutations, deletions,
amplifications, chromosomal rearrangements of the miRNA-encoding DNA regions
or epigenetic changes, leading to aberrant miRNA expression \citep{Calin2006}.
miRNA function can also be altered by abnormalities in the miRNA-processing
machinery (e.g. DICEROIREYHTYMÄ) and by mutations in target mRNA sequences.

Studies have shown that microRNAs have important roles in tumor initiation,
progression and metastasis \citep{Lin2015}. MicroRNA expression signatures
also correlate with numerous cancer features, such as diagnosis, staging,
tissue, progression, prognosis and treatment response, and all studied cancers
have had miRNA expression profiles differing from healthy tissue
\citep{Calin2006}. In fact, microRNAs appear to be globally downregulated in
cancers \citep{Lu2005}. Therefore, it seems clear that microRNAs are involved
in many of the pathways regulating cancer development and progression, and it
has been shown that the deregulation of even a single miRNA can lead to cancer
\citep{Costinean2006}.

CELL-FREE MIRNA MARKKERINA. POTENTIAALISIA HOITOKEINOJA.










\subsection{Cancer}\label{cancer}

Cancer is a disease of uncontrolled overgrowth of a population of cells. It is
generally viewed as a genetic disease, albeit mostly not inherited, as it is
caused by  mutations in the tumor genome. These mutations cause malfunction
and dysregulation of the genetic machinery regulating cellular functions, such
as cell proliferation, differentiation and apoptosis, resulting in
unregulated growth and malignant tumor formation.

There are several classes of genes that influence tumor growth, the two main
categories being oncogenes and tumor suppressors. Oncogenes were first
identified in retroviruses and later shown to be proto-oncogenes, which by
mutation develop into oncogenes whose over activity promotes tumor growth
\citep{Varmus1988}. Tumor suppressor genes are often regulators of cell
proliferation or other so called housekeeping genes that work to ensure the
proper functioning of cells and the apoptosis of misbehaving ones. The
inactivation of these genes then leads to tumor progression. The existence of
tumor suppressors was first hypothesized by Alfred Knudson, who formed the “two-
hit hypothesis” while studying the epidemiology of retinoblastoma
\citep{Knudson1971}. He suggested that, for cancer to develop, both copies of
a tumor suppressor gene should become inactivated and that in inherited
cancers one mutation is acquired in the germline and the other occurs in
somatic cells, whereas in sporadic cancers both mutations happen in somatic
cells.

The idea of oncogenes and tumor suppressor genes was later expanded on by
Douglas Hanahan and Robert Weinberg in their seminal article The Hallmarks of
Cancer \citep{Hanahan2000}. The hallmarks are a set of six features which
tumors often acquire to become malignant. The features are: sustaining
proliferative signaling, evading growth suppressors, resisting cell death,
enabling replicative immortality, inducing angiogenesis, and activating
invasion and metastasis. Weinberg and Hanahan postulated that at least three
of these six features are required for invasive cancer to develop.

Recently, Hanahan and Weinberg updated the hallmarks with two new upcoming
hallmarks and two enabling characteristics \citep{Hanahan2011}. The new
hallmarks are deregulation of cellular energetics and avoiding immune
destruction. The enabling characteristics of malignant tumors are genome
instability and mutation, and tumor-promoting inflammation through recruition
of the immune system. Genome instability and mutation is of special importance
as much of cancer and tumor research focuses  on identifying mutated or
aberrantly expressed genes that promote tumor progression.



\subsubsection{Breast cancer}\label{breast-cancer}

Breast cancer constitutes a significant health issue globally. It is the most
common cancer in women and the second most common cancer overall;
approximately 1.7 million women develop breast cancer annually world-wide, and
in 2012 there were 522 000 breast cancer related deaths.
\citep{Ferlay2015} In Finland there are approximately 4 800 new cases of
breast cancer annually, and there were 874 breast cancer related deaths in
2013 \citep{Syoparekisteri}.

Most breast cancers are sporadic; only 5-7\% of breast cancer cases are of
familial type \citep{Melchor2013}. However, 15-30\% of breast cancer patients
have a family member or relative with breast cancer \textbf{VIITE?}. This is due to the high
frequency of breast cancer in many western populations, but also suggests that
there are unknown genetic factors and environmental factors that have an
impact on breast cancer development.  Indeed, breast cancer is a hormone
related disease and hormonal factors are known to have an impact on
breast cancer risk.


\subsubsection{Breast cancer classification}\label{breast-cancer-classification}

The basic classification of cancer is based on the site -- that is, the organ or
tissue, such as breast or colon -- where the primary tumor develops. Cancers
occurring in each site are, however, heterogeneous in their nature and have
different behavior and prognosis. The heterogeneous cancers
within each organ are secondarily classified by morphology, the microscopic structure of
the cancer tissue.

The morphological classification of breast cancer is based
on the WHO classification from 2003 and includes altogether 19 histological
subtypes of invasive breast cancer \citep{Tavassoli2003,Weigelt2009}. Of these invasive ductal carcinoma
not otherwise specified (IDC NOS), accounts for the by far largest
histological group.  Additionally, the WHO classification includes the TNM
classification, characterization of the primary tumor (T), lymph node status
(N) and distant metastasis (M) and stage grouping based on the TNM data. All
invasive cancers are also graded into well (1), moderately (2) or poorly (3)
differentiated tumors based on  tree features; tubule formation as an
expression of glandular differentiation, nuclear pleomorphism and mitotic
counts \citep{Tavassoli2003}.

In the clinic many other tumor characteristics are used. These
include the age of patient, lympho-vascular invasion as well as expression of
estrogen (ER) and progesterone receptors (PR), and Her2, which are routinely
studied for breast cancers. Together these data can be used to group patients
into risk categories for prognosis and choice of treatment using for example
StGallen criteria \citep{Goldhirsch2007} or NIH criteria \citep{Eifel2001} or others.

One of the problems with morphological classification of breast tumors is that
over 50\% do not show any particular features and are classified as IDC NOS,
in spite of the fact that tumors in this large group have very different
clinical outcomes. More recently, expression profiling has led to a suggestion
of new classification of breast cancers \citep{Perou2000,Sorlie2001}. By studying 65
samples from breast tumors with expression profiling Perou et al.
distinguished four subgroups based on gene expression, namely ER+/luminal-
like, basal-like, Erb-B2+ and normal breast \citep{Perou2000}.

Expression profiling has also led to development of new prognostic tests to
help to determine the need of adjuvant chemotherapy.  These tests include
OncotypeDx, 21-Gene recurrence score, MammaPrint a 70-Gene test and PAM50, a
50-Gene test, and have been evaluated by several groups and suggested to be
valid and promising, but their utility in clinical decision making is still
unclear \citep{Azim2013}. The PAM50 molecular classification ... \textbf{TÄSTÄ LISÄÄ?}

%Later Haibe-Kains et al have shown that the expression based classification of
%breast cancer into four categories can essentially be achieved by studying
%three major genes \citep{T16} ...



\subsubsection{MicroRNAs in breast cancer}








\subsection{Measurement of gene expression}\label{measurement-of-gene-expression}

\begin{itemize}
  \item blotting, qPCR
  \item microarrays
  \begin{itemize}
    \item two-color arrays
    \item oligonucleotide arrays
  \end{itemize}
  \item sequencing-based methods
  \item microarray analysis
  \begin{itemize}
    \item preprocessing and normalization
    \item probe annotation problems
  \end{itemize}
  \item protein arrays (RPPA)
\end{itemize}


\subsubsection{Microarrays}


\subsubsection{MicroRNA detection}

The same methods that have been employed for measuring mRNA (i.e. gene)
expression are generally applicable for measuring microRNA expression as well,
ranging from the initial discovery of lin-4 with northern blotting
\citep{Lee1993} to microarrays \citep{Liu2008} and recent studies using 
next generation sequencing (NGS)
\textbf{viite?} \citep{esimerkki}. However, as Hunt and colleagues in
a recent and thorough review point out, there are several
challenges in detecting miRNAs in particular \citep{Hunt2015}.

MicroRNAs are very short and only comprise approximately 0.01\% of RNA
typically extracted from a sample \citep{Dong2013}. This implies that miRNA
detection must be highly sensitive and emphasizes the importance of proper
sample preparation and extraction. MicroRNAs from the same family can differ by only one
base, which in turn requires high specifity to be able to distinguish between
family members. On the other hand, variation in miRNA processing
can result in slight sequence variations, or isoforms, of a single miRNA,
also known as isomiRs \citep{StaregaRoslan2011,Lee2010}. This means high specifity
or an incorrect reference sequence (e.g. that of a lowly-expressed isomiR) 
used for detection can cause inaccurate measurements. IsomiRs may also
have different functions resulting from altered target specifity \citep{Chugh2012}.
The existence of the pri-miRNA, pre-miRNA and mature miRNA molecules provides an additional
challenge for measurement methods.
%, although differentiation between these maturation stages is not necessarily required.

Many of the challenges mentioned are solved by NGS, as sequencing can detect
variations of even one nucleotide and does not necessarily depend on
previously identified sequences. However, not all
identified short RNAs are functional miRNAs, and NGS conveys its own set of
problems relating to high cost, significant computational complexity and
validation efforts to distinguish relevant data from noise
\citep{Chugh2012,Hunt2015}.







\subsection{MicroRNA target prediction}\label{mirna-target-prediction}

This section presents computational methods that have been used to predict
putative target genes for microRNAs and regulatory networks between genes and
microRNAs.





\subsubsection{Sequence-based target prediction}\label{sequence-based-target-
prediction}

\textbf{SELITÄ TÄÄ HOMMA JA ERI ALGORITMIT.}

Evolutionally conserved targets can be simply identified by searching for
matches to the seed sequence of a miRNA in mRNA 3' UTR's and then checking if
these matching sites are conserved in orthologous 3' UTR's of other species.
Conserved matches are considered predicted regulatory sites and, hence, miRNA
targets \citep{Bartel2009}. All sequence-based target prediction algorithms are basically
refinements of this basic protocol. This also leads to the notion of miRNA
families, which are mature miRNAs that share the same nucleotides at positions
2-8. All miRNAs within a family have the same predicted targets based on sequence.
\textbf{(Osa tästä kpl:sta miRNA genomicsiin?)}

Sequence-based prediction suffers from two major drawbacks. First, there are
high numbers of false positives \textbf{SYY JA VIITE
\citep{Sethupathy2006?}}. Second, the predictions are static and do not
account for different tissues or disease states.

- A seed match does not always confer repression by the matching miRNA.\citep{Grimson2007} %MicroRNA targeting specifity in mammals


\paragraph{MicroRNA target databases}





\subsubsection{Integration of expression data with sequence data}\label
{integration-of-expression-data-with-sequence-data}

Recently, a plethora of new methdos have been published that integrate
sequence-based target prediction with expression data. This helps combat the
high false-positive rate of sequence-only methods and enables tissue and
disease specific support for the predictions in real-world data. Recent
evidence indicates that miRNAs act predominantly through degradation
\citep{CITE}. Thus, it is feasible to use miRNA and mRNA expression data to
infer target relationships, since the regulatory effect of miRNAs should be
directly reflected in mRNA levels. All proposed methods use sequence-based
predictions as a starting point by considering only miRNA-mRNA pairs predicted
by at least one sequence-based prediction algorithm. This section provides a
review of algorithms for integrating miRNA and mRNA expression and
implementations in select target prediction methods.

\textbf{KOLME KATEGORIAA: KORRELAATIO, REGRESSIO, BAYESIAN?}



\paragraph{Correlation methods (incl MI)}\label{correlation-methods}

Mutual information (MI) is a simple measure of similarity between two
variables. \textbf{SELITÄ MI TARKEMMIN JA KAAVAN KANSSA JA LÄHDE} Thus, MI can
be used to measure the interdependence of miRNA-mRNA pairs from expression
data. However, MI does not distinguish the direction of the interaction, which
is highly relevant for miRNAs that are believed to mostly downregulate mRNA
expression. This constitutes a major drawback.

Correlation \textbf{SELITÄ KORRELAATIO JA LÄHDE}.

MAGIA \citep{Sales2010} is a webservice that implements both the MI and
correlation approaches. It also constructs a bipartite network of the top 250
predicted miRNA-mRNA pairs and provides links to several databases for further
examination and validation of results.

\textbf{VAI LAITTAISKO MAGIAN JO MI:N YHTEYTEEN JA TOISTAA VAAN
KORRELAATIOSSA?}



\paragraph{Regression methods}\label{regression-methods}

\textbf{SELITÄ REGRESSIO.}

Engelmann et al used least angle regression to show that gene expression can
be predicted from miRNA expression \citep{Engelmann}.

\textbf{SELITÄ REGULARISOIDUT/SHRINKAGE-TYYPPISET REGRESSIOMALLIT.}

While aiding with interpretability, shrinkage also has several drawbacks.
First, only a limited number of covariates may be included in the model, and
thus some relevant associations can be missed by number of included covariates
alone \citep{vanIterson2013}. Second, shrinkage may remove covariates highly
associated with and functionally regulating the response, instead retaining an
uninvolved covariate that correlates with actual regulators \citep{Engelmann}.
Relating to both limitations, van Iterson et al showed for one dataset that
lasso did not consistently select highly correlated miRNA-mRNA pairs
\citep{vanIterson2013}.

\textbf{SAISKO NÄIHIN VIITTEET REGRESSIOTEORIASTA EIKÄ TOSTA PAPERISTA?}


\subparagraph{Global test (regression)}\label{global-test-regression}

van Iterson et al recently proposed a miRNA target prediction method based on
the global test \citep{vanIterson2013}. \textbf{SELITÄ GLOBAL TEST}.

The method by van Iterson et al uses TargetScan, microCosm and PITA for
putative sequence-based targets and is available as an R package called
miRNAmRNA \citep{vanItersonWeb}.





\subsubsection{Bayesian methods}\label{bayesian-methods}

First published Bayesian analysis by Thomas Bayes in 1763 \citep{Gelman2013}.

Advantages of Bayesian methods:
\begin{itemize}
  \item
  quantification of uncertainty as probabilities, knowledge about anything
  unknown is described as a probability distribution, easy to comprehend
  \item
  common-sense interpretation of credible intervals compared to frequentist
  confidence intervals
  \item
  flexibility allows constructing complex models with relative ease (e.g.
  hierarchical models)
  \item
  Adding more data sequentially is possible by using the previous posterior
  distribution as the new prior distribution. This can be especially useful in
  a clinical research context, where data are often collected within a long
  timespan.
\end{itemize}
The challenge in Bayesian analysis is setting up proper probability models for
the parameters and observations. \citep{Gelman2013} This includes the prior
distributions of the parameters as well as the likelihood of the observed data.
The choice of prior is also a source of much controversy, as it is based
on experience and reasoning of the statistician.

The posterior distribution represents a compromise between the prior (and
hence, prior information) and the observed data, with the data having an
increasing effect as the sample size increases. \citep{Gelman2013}.

\textbf{OMIN SANOIN: Confidence intervals work their best when you don't know much about a
parameter beyond the information contained in a data set. And further,
credibility intervals won't be able to improve much on confidence intervals
unless there is prior information which the confidence interval can't take
into account, or finding the sufficient and ancillary statistics is hard.}

- The posterior distribution also provides a more comprehensive view of
one's knowledge on the parameter of interest than, say, a single confidence
interval (or even a series of confidence intervals).
- In many instances, using a non-informative prior results in similar or
equal results than frequentist analysis, but the strength of Bayesian
analysis comes from including prior knowledge in the prior distribution. \citep{Jaynes?}

The frequentist approach only considers the data to have a probability
distribution, the likelihood. The process giving rise to the data, and the
parameters that define it, are considered fixed. The observed data are
assessed with respect to other data generated by the same model.




\paragraph{Simulation}\label{simulation}

Bayesian analysis often involves simulation if the form of sampling from the
obtained posterior distribution. This is convenient -- and necessary -- when
the exact probability density function cannot be explicitly obtained through
integration. Additionally, simulation often has the advantage of pointing out
problems in the model specification when simulated values are extremely small
of large.

Simulation methods:
\begin{itemize}
  \item
  sampling from probability distributions (easy with modern pseudorandom
  number generators)
  \item
  \textbf{mikäseyksinkertasinonkaan}
  \item
  Gibbs
  \item
  Hamiltonian Monte Carlo (this is used by Stan)
\end{itemize}


\paragraph{Bayesian regression}\label{bayesian-regression}

\textbf{SELITÄ MITEN BAYES-REGRESSIO EROAA TAVALLISESTA.}
