%!TEX root = dippa.tex
%%% This file contains the discussion section of my master's thesis.
%%% Author: Viljami Aittomäki

\section{Discussion}

This thesis presented a review of literature concerning microRNAs and the
prediction of microRNA target genes. A modern Bayesian regression model and
variable selection method was applied to predict protein expression from mRNA
and miRNA expression in breast cancer tumor samples with the goal of
identifying putative miRNA target genes.

\paragraph{Overview of results}

Correlation between mRNA and protein expression was poor on average, a finding
supported by many previous studies \citep{Payne2015}. Interestingly,
there was no difference between correlations of predicted or validated
miRNA-target pairs or randomly chosen ones and all of these correlation
distributions were zero centered. This illustrates the
complexity of expression regulation and supports the view that modeling
single interaction pairs individually is unlikely to be sufficient in
general.

Aure et al used Lasso regression for a similar analysis
%prediction of protein expression from mRNA and miRNA expression 
to identify miRNAs significantly
affecting protein expression in breast cancer. They used a multi-step
process, where miRNAs input into the Lasso regression were first
filtered with simple linear regression, only miRNAs deemed significant
in the univariate regression model were included. This approach is
flawed in the sense, that it does not have the advantage of multivariate
models to identify singly weak but combinatorially strong effects, since univariate
modeling is used as a filtering step.

\paragraph{Disadvantages and improvement}

An obvious way to improve the model would be to include sequence-based target
information. Bayesian modeling provides a formal and relatively easy way to
achieve this with hierarchical models. MicroRNAs that that are putative
regulators of a gene based on sequence could have different (more slack)
priors or superpriors than miRNAs without seed sequence matches. Furthermore,
experimentally validated regulators could have different priors as well.

One disadvantage of the proposed method relates to scaling of the regression
covariates, that is, the mRNA and miRNA expression variables. Sequencing studies have shown
that a relatively small number of miRNAs accounts for over 80\% of tissue microRNA
\citep{Landgraf2007}. Therefore, changes in the expression of these highly
abundant miRNAs may have a relatively large impact on protein levels, where as
similar changes in less expressed miRNAs may have little to no efect on target gene
expression. This difference is lost by scaling all miRNA variables to a similar scale.

However, many of the most abundant miRNAs are ubiquitously expressed across
different tissues \citep{Landgraf2007} and, therefore, possibly less
interesting with regards to disease pathogenesis. Additionally, inclusion of
the gene expression measurement in the regression necessitates some form of
scaling, as mRNA and miRNA expression profiles are measured with different platforms
and processed using different algorithms and, therefore, are not directly
comparable. One could easily envision a scaling procedure where the relative
levels of different miRNAs are preserved and gene expression is scaled
relative to mean miRNA expression, for example, but it is debatable whether
this would be appropriate either.

Approximately half of the predicted interactions were positive. Some of these
could indicate indirect regulation. However, this proportion seems much too
high, as most known microRNA interactions are suppressive. In fact, of all the
experimentally validated human miRNA targets listed in DIANA-TarBase, only
around 0.2\% report positive regulation by the miRNA. Therefore, many of the
predicted positive interactions are possibly false positives. To amend this,
the model could easily be restricted to only negative interactions (using a
nonpositive prior for $\beta$), and this has previously been reported to
increase prediction accuracy \citep{Muniategui2013}.

The proposed model also does not account for the fact that microRNAs have
several, even hundreds, of target transcripts \citep{}. Therefore, the regulatory
effect of a single miRNA is most likely spread across several genes. In
combination with transcripts having several regulating miRNAs, this many-to-many
nature of microRNA regulation ultimately calls for computational methods
that model the whole regulatory network at once. This, however, becomes a much
more difficult problem than simple multiple linear regression. \textbf{Akilla
oli tähän joku metodiehdotus, johon voisi viitata.}

\paragraph{Future prospects}

The recent development of CLIPseq and similar methods has made high-throughput
experimental microRNA target discovery possible, partially replacing the need
for computational target prediction. Nonetheless, experimental methods
(particularily high-throughput ones) are not immune to error, and microRNA
regulation is complex with many unconventional regulatory relationships and
mechanisms haveing been discovered. Integrative computational approaches --
combining several levels of data -- beyond correlation will, thus, remain
important in the future. The elucidation of complex regulatory networks using
network-level modeling is becoming feasible with modern experimental and
computational methods. Adapting this approach will be essential, as it has the
ability to better capture the true biology of the regulation of gene
expression.

Many aspects of microRNA biology and function are still unknown, and they 
remain an exciting avenue of research. \textbf{HOITOJA JA SEMMOSTA.}