%!TEX root = dippa.tex
%%% This file contains the discussion section of my master's thesis.
%%% Author: Viljami Aittom√§ki

\section{Discussion}

This thesis presented a review of the basics of gene expression, microRNAs and
the computational prediction of microRNA target genes. A modern Bayesian
variable selection method was applied in the context of regression, to predict
protein expression from mRNA and miRNA expression in breast cancer tumor
samples, with the goal of identifying putative miRNA target genes in breast
cancer. The Bayesian method was compared to lasso regression, a popular method
for target prediction from expression data.


\subsection*{Overview of results}

Correlation between mRNA and protein expression was mostly low, a finding
supported by many previous studies \citep{Payne2015}. Interestingly, there was
no virtually difference between correlations of validated miRNA-target pairs
or randomly chosen ones. This illustrates the complexity of expression
regulation and supports the view that modeling single interaction pairs
individually is unlikely to be sufficient in general.

Compared to lasso regression PPVS achieved better model fit, yet from a target
prediction point of view, performance of the two methods was similar.
There was little overlap of predictions made by the two methods, a common
issue in microRNA target prediction. Only a small fraction of predicted
targets were validated according to TarBase, but it should be noted, that is
probably true in general of all miRNA targets; only a limited number of
validation studies have been done.

Aure et al used lasso regression for a similar analysis of the same dataset
%prediction of protein expression from mRNA and miRNA expression 
to identify miRNAs significantly affecting protein expression in breast cancer
\citep{Aure2015}. They used a multi-step process, where miRNAs deemed
significant in a simple univariate regression model were used as input in
lasso regression. This approach is flawed in the sense, that it loses some of
the power of multivariate models to identify singly weak but combinatorially
strong effects, since univariate modeling is used as a filtering step. It also
uses the same data twice, causing bias.


\subsection*{Possible improvements}

\textbf{KERTOIMIEN RAJOITTAMINEN NEGATIIVISIKSI!}

An obvious way to improve the model would be to include sequence-based target
information. Bayesian modeling provides a formal and relatively easy way to
achieve this with hierarchical models. MicroRNAs that that are putative
regulators of a gene based on sequence could have priors (or hyperpriors) with
less regularization than miRNAs without sequence matches. However, as
the authors of GenMir noted, including sequence features did not result
in a significant improvement of their method \citep{Huang2007}.

One disadvantage of the proposed method relates to scaling of the regression
covariates, that is, the mRNA and miRNA expression variables. Sequencing studies have shown
that a relatively small number of miRNAs accounts for over 80\% of tissue microRNA
\citep{Landgraf2007}. Therefore, changes in the expression of these highly
abundant miRNAs may have a relatively large impact on protein levels, where as
similar changes in less expressed miRNAs may have little to no effect on target gene
expression. This difference is lost by scaling all miRNA variables to a similar scale.

However, many of the most abundant miRNAs are ubiquitously expressed across
different tissues \citep{Landgraf2007} and, therefore, possibly less
interesting with regards to disease pathogenesis. Additionally, inclusion of
the gene expression measurement in the regression necessitates some form of
scaling, as mRNA and miRNA expression profiles are measured with different platforms
and processed using different algorithms and, therefore, are not directly
comparable. One could easily envision a scaling procedure where the relative
levels of different miRNAs are preserved and gene expression is scaled
relative to mean miRNA expression, for example, but it is debatable whether
this would be appropriate either.

Approximately half of the predicted interactions were positive. Some of these
could indicate indirect regulation. However, this proportion seems much too
high, as most known microRNA interactions are suppressive. In fact, of all the
experimentally validated human miRNA targets listed in DIANA-TarBase, only
around 0.2\% report positive regulation by the miRNA. Therefore, many of the
predicted positive interactions are possibly false positives. To amend this,
the model could easily be restricted to only negative interactions (using a
non-positive prior for $\beta$), and this has previously been reported to
increase prediction accuracy \citep{Muniategui2013}.

The proposed model also does not account for the fact that microRNAs have
several, even hundreds, of target transcripts \citep{}. Therefore, the regulatory
effect of a single miRNA is most likely spread across several genes. In
combination with transcripts having several regulating miRNAs, this many-to-many
nature of microRNA regulation ultimately calls for computational methods
that model the whole regulatory network at once. This, however, becomes a much
more difficult problem than multivariate linear regression.

In conclusion, the work in this thesis shows that the proposed method of
projection predictive variable selection is applicable to microRNA target
predction. However, further refinements to the model are warranted to improve
preformance. In the presented form, compared to a simpler alternative, the
method offered only a small advantage from a modeling perspective, and
apparent advantage from a biological perspective , but incurred a large
computational burden. The choice of model-size parameters $\alpha$ and
$\gamma$, and therefore the choice of final model size and ultimately the
model itself, proved untrivial. A data-driven approach for optimizing the
parameter values would perhaps be of value.


\subsection*{Future prospects}

The recent development of CLIPseq and similar methods has made high-throughput
experimental microRNA target discovery possible, partially replacing the need
for computational target prediction. Nonetheless, experimental methods
(particularly high-throughput ones) are not immune to error, and gene
regulation is vastly complex with many unconventional regulatory mechanisms
having been discovered. Integrative computational approaches --
combining several levels and types of data -- beyond correlation will, thus, remain
important in the future.

The elucidation of complex regulatory networks using
network-level modeling is becoming feasible with modern experimental and
computational methods. Employing this approach will be essential, as it has the
ability to better capture the true nature of gene regulation and cellular
biology.

Many aspects of microRNA biology and function are still unknown. As well as
helping us understand the complexities of molecular cell biology, uncovering
microRNA function also offers interesting possibilities in diagnostics and even
treatment of cancer, as has already been shown for instance for breast
cancer. MicroRNAs will, therefore, remain an exciting avenue of research in the future.
