

\section{Discussion}

This thesis presented a review of literature concerning microRNAs and
prediction of microRNA target genes. A Bayesian regression model was applied
to predict protein expression from mRNA and miRNA expression in breast cancer
tumor samples.

\paragraph{Overview of results}

Correlation between mRNA and protein expression was poor on average, a finding
supported by many previous studies \citep{Payne2015}. Interestingly,
there was no difference between correlations of predicted or validated
miRNA-target pairs or randomly chosen ones and all of these correlation
distributions were zero centered. This illustrates the
complexity of expression regulation and supports the view that modeling
single interaction pairs individually is unlikely to be sufficient in
general.

\paragraph{Disadvantages and improvement}

An obvious way to improve the model would be to include sequence-based target
information. Bayesian modeling provides a formal and relatively easy way to
achieve this with hierarchical models. MicroRNAs that that are putative
regulators of a gene based on sequence could have different (more slack)
priors or superpriors than miRNAs without seed sequence matches. Furthermore,
experimentally validated regulators could have different priors as well.

One disadvantage of the proposed method relates to scaling of the regression
covariates, that is, the mRNA and miRNA expression variables. Sequencing studies have shown
that a relatively small number of miRNAs accounts for over 80\% of tissue microRNA
\citep{Landgraf2007}. Therefore, small changes in the expression of these highly
abundant miRNAs may have a relatively large impact on protein levels, where as
changes in less expressed miRNAs may have little to no efect on target gene
expression. This difference is lost by scaling all miRNA variables to a similar scale.

However, many of the most abundant miRNAs are ubiquitously expressed across
different tissues \citep{Landgraf2007} and, therefore, possibly less
interesting with regards to disease pathogenesis. Additionally, inclusion of
the gene expression measurement in the regression necessitates some form of
scaling, as mRNA and miRNA expression profiles are measured with different platforms
and processed using different algorithms and, therefore, are not directly
comparable. One could easily envision a scaling procedure where the relative
levels of different miRNAs are preserved and gene expression is scaled
relative to mean miRNA expression, for example, but it is debatable whether
this would be appropriate either.

The proposed model also does not account for the fact that microRNAs have
several, even hundreds, of target transcripts \citep{}. Therefore, the regulatory
effect of a single miRNA is most likely spread across several genes. In
combination with transcripts having several regulating miRNAs, this many-to-many
nature of microRNA regulation ultimately calls for computational methods
that model the whole regulatory network at once. This, however, becomes a much
more difficult problem than simple multiple linear regression. \textbf{Akilla
oli tähän joku metodiehdotus, johon voisi viitata.}

\paragraph{Future prospects}

\begin{itemize}
	\item miRNAs as potential treatments
	\item data integration on several levels important
	\item NGS solves some problems
	\begin{itemize}
		\item CLIP already does high-throuput experimental identification of MTIs?
	\end{itemize}
\end{itemize}

