%!TEX root = dippa.tex
%%% This file contains the discussion section of my master's thesis.
%%% Author: Viljami Aittom√§ki

\section{Discussion}

This thesis presents a review of the basics of gene expression, microRNAs and
the computational prediction of microRNA target genes. A modern Bayesian
variable selection method was applied in the context of regression, to predict
protein expression from mRNA and miRNA expression in breast cancer tumor
samples, with the goal of identifying putative miRNA target. The Bayesian
method was compared to lasso regression, a popular method for target
prediction from expression data.

Quality control plots suggested the presence of noise in the data, especially
in the miRNA microarrays. More strict filtering of miRNA data could lead to
better overall results. The choice of data model can also alleviate the
effects of noise. The
student-$t$ distribution, which has heavier tails than the normal
distribution, would likely be a better choice for the regression model
likelihood. This, however, would require a different solution to the
projection Eq. \eqref{eq:projection} than the one derived by Piironen
and Vehtari \citep{Piironen2015} and, therefore, was not tried in this work.

Correlation between mRNA and protein measurements for the same gene were
mostly low, a finding supported by previous studies \citep{Payne2015}.
Interestingly, there was practically no difference between correlations of
validated miRNA-target pairs or randomly chosen ones. This supports the view
that modeling single interaction pairs individually is unlikely to be
sufficient for effective target prediction using expression data.

It is possible that low correlation of miRNA-target pairs was mostly due to the
dataset used and measurement noise it contained, however, 
%as discussed in Section \ref{expr-methods},
correlation has low power to detect weak relationships and
cannot capitalize on combinatorial effects. The low correlations also suggest
that simple (Pearson) correlation of expression data is possibly inadequate for
finding miRNA targets. Poor prediction results using correlation have been
reported before (see for example \citep{Muniategui2012}).

There could be several explanations for the increasingly poor performance of
larger models, evident in Figure \ref{fig:n-miRNAs-vs-R2}. The largest models
could be a result of the covariates not explaining the outcome variable well.
This results in a large model, as the inclusion of each covariate can provide
only a minute improvement to submodel performance and, therefore, a large
number of covariates is required to satisfy the criterion for choosing model
size. The assumption that the reference model represents the best current knowledge
could also be false, as seemed to be the case for some genes where the
submodels performed consistently better than the full model. This likely means
that the reference model has overfit the training data (in the cross-
validation).

From a biological perspective, poor performance means that the microRNAs did
not provide additional information for predicting protein expression after
accounting for gene expression. This could be due to relevant miRNAs missing
from the dataset, the number of observations being too small leading to low
statistical power (unable to capture the often small effect that miRNAs have),
or the biological heterogeneity of breast cancer.
% a gene truly not having
% miRNA regulators in breast tissue, insufficient data (the true regulators of a
% given gene might be missing from the data)
The proposed regression model could also be inadequate for capturing the actual	
biological effect of miRNAs, though previous research seems to suggest otherwise.


% Secondly, the marginal posterior of a single predictor can
% indicate non-significance, while the joint posterior of several predictors
% combined might still achieve significance. This would indicate that the effect
% of miRNAs is only significant when acting simultaneously, a hypothesis that is
% supported by experimental evidence, as discussed in section
% \ref{microrna-function}.

Compared to lasso regression PPVS achieved better model fit, yet from a target
prediction perspective, performance of the two methods was similar. There
was little overlap of predictions made by the two methods, a common issue in
microRNA target prediction \citep{vanIterson2013}. Only a small fraction of
predicted targets were validated according to TarBase and miRTarBase, however,
this is probably true of all miRNA targets in general; only a
limited number of validation studies have been published. The computational cost
of PPVS was significantly higher than that of lasso: MCMC simulations took hours
compared to less than a minute per model for lasso.

Approximately half of the regression coefficients for miRNAs were positive,
suggesting that those miRNAs increase gene expression. Some of these
could indicate indirect regulation. However, this proportion seems too
high, as the vast majority of known microRNA interactions are suppressive. In
fact, of all the experimentally validated human miRNA targets listed in
TarBase, only approximately 0.2\% show positive regulation by the miRNA.
Therefore, many of the predicted activating interactions are possibly false
findings. They could be caused by miRNA expression mirroring the involvement of other
regulatory factors not included in the data. To correct for this, the model could
easily be restricted to only negative interactions (using a non-positive prior
for $\beta$). This has previously been reported to increase prediction
performance \citep{Muniategui2013}.

Previous studies have shown that microRNA signatures correlate
with different breast cancer subtypes \citep{Blenkiron2007}. This suggests that
using pooled datasets of various tumors, such as the data used in this
study, is likely to miss subtype-specific miRNA effects, unless this is accounted
for in the model. This could be achieved in the proposed method by
constructing a hierarchical model that includes tumor-subtype data.

Another way to improve the proposed model would be to include sequence-based
target information, as most published methods do. This could be achieved with
indicator variables, a weighting scheme, or more elaborately by including
sequence-based data within the hierarchical-shrinkage prior to impose less
regularization on putative target pairs. However, as the authors of GenMir
noted, including sequence features did not result in a significant improvement
of their method \citep{Huang2008}.\footnote{It should be noted, that GenMir
uses sequence-based predictions as a preliminary filter step. Therefore,
it is perhaps not surprising that including the same type of data within the
model does not produce substantial improvement.}

% One disadvantage of the proposed method relates to scaling of the regression
% covariates, that is, the mRNA and miRNA expression variables. Sequencing studies have shown
% that a relatively small number of miRNAs accounts for over 80\% of tissue microRNA
% \citep{Landgraf2007}. Therefore, changes in the expression of these highly
% abundant miRNAs may have a relatively large impact on protein levels, where as
% similar changes in less expressed miRNAs may have little to no effect on target gene
% expression. This difference is lost by scaling all miRNA variables to a similar scale.

% However, many of the most abundant miRNAs are ubiquitously expressed across
% different tissues \citep{Landgraf2007} and, therefore, possibly less
% interesting with regards to disease pathogenesis. Additionally, inclusion of
% the gene expression measurement in the regression necessitates some form of
% scaling, as mRNA and miRNA expression profiles are measured with different platforms
% and processed using different algorithms and, therefore, are not directly
% comparable. One could easily envision a scaling procedure where the relative
% levels of different miRNAs are preserved and gene expression is scaled
% relative to mean miRNA expression, for example, but it is debatable whether
% this would be appropriate either.

The proposed model does not account for the fact that microRNAs have
several, even hundreds, of target transcripts \citep{Friedman2009}. Therefore, the
regulatory effect of a single miRNA is most likely spread across several
genes. In combination with transcripts having several regulating miRNAs, this
many-to-many nature of microRNA regulation ultimately calls for computational
methods that model the whole regulatory network at once, such as regression
models with multivariate targets. This, however, becomes a much more difficult
problem than multivariate linear regression.

Aure et al \citep{Aure2015} used lasso regression for a similar analysis of the same dataset.
%prediction of protein expression from mRNA and miRNA expression 
%to identify miRNAs significantly affecting protein expression in breast cancer
They used a multi-step process, where only miRNAs deemed
significant in a univariate regression model were used as input in multivariate
lasso regression. This approach is flawed in the sense, that it loses some of
the power of multivariate models to identify singly weak but combinatorially
strong effects, as univariate modeling is used as a filtering step. It also
effectively uses the same data twice, causing bias, and introduces a
multiple hypothesis testing problem. Therefore, a multivariate
approach (such as the one presented here or earlier ones with slight modification)
would likely be preferable.

In conclusion, the work in this thesis shows that the proposed method of
projection predictive variable selection is applicable to microRNA target
prediction. However, further refinements to the model are warranted to improve
performance. In the presented form, compared to a simpler alternative, the
method offered only a limited advantage from a modeling perspective, and no
apparent advantage from a biological perspective, but incurred a large
computational burden. The choice of parameters $\alpha$ and $\gamma$, which
define the threshold for model size, proved nontrivial. The values chosen
had a large impact on the sizes of resulting models and, therefore,
a data-driven approach for optimizing the parameter values would perhaps be useful.




\subsection*{Future prospects}

The recent development of CLIP-seq and similar methods has made high-throughput
experimental microRNA target discovery possible, partially
replacing the need for computational target prediction. Nonetheless,
experimental (particularly high-throughput) methods are not immune to error,
and gene regulation is vastly complex with many unconventional regulatory
mechanisms having been discovered. Integrative computational approaches
beyond correlation -- combining several types of data -- will, thus,
remain important in the future. Possibilities for integrating data include
incorporating copy number variation, other regulatory RNAs, transcription factors,
other protein-level regulatory factors such as phosphorylation, and
epigenetic mechanisms into models.

The elucidation of complex regulatory networks using network-level modeling is
becoming feasible with modern experimental and computational methods.
Employing this approach will be essential, as it has the ability to better
capture the true nature of gene regulation and cellular biology.

Many aspects of microRNA biology and function still remain unknown. Uncovering
miRNA function offers interesting possibilities in diagnostics and treatment
of disease, and will further our understanding of the complexities of
molecular cell biology. Therefore, microRNAs remain an exciting avenue
of research.
