%!TEX root = dippa.tex
%%% This file contains Introduction of my master's thesis.
%%% Author: Viljami Aittom√§ki

\section{Introduction}
%% No page number etc for first page
\thispagestyle{empty}

MicroRNAs (miRNAs) are short single-stranded RNA-molecules, that act in post-
transcriptional regulation of gene expression \citep{Bartel2004}. They have
been found in a wide variety of animals and plants, and also in viruses.
MicroRNAs are highly conserved in evolution and function in diverse
developmental, physiological and pathological processes. miRNAs have been
indicated as having an important role in the formation of numerous diseases,
including several types of cancer \citep{Calin2006}. Therefore, the study of
microRNAs and their function in cancer can offer insights into tumorigenesis
and cancer progression as well as potential new biomarkers and treatments.

Identifying the target genes regulated by microRNAs is key to understanding their
function, both in cellular physiology and pathology. Experimental laboratory
studies to identify miRNA targets are both laborious and costly. In fact,
finding all miRNA targets by experimental studies alone is unfeasible, considering
that any gene is potentially targeted by any miRNA, giving rise to tens of
millions of potential interactions.

A plethora of methods for computational identification of microRNA target
genes have, thus, been developed \citep{Muniategui2013}. Early methods were based on sequence
similarity of the microRNA and messenger RNA (mRNA) of putative target genes.
More recent methods compare the expression profiles of miRNAs and mRNAs in
cell cultures or tissue samples to elucidate interactions transpiring in
cells. Most of these expression-based methods rely on variations of
correlation or multivariate linear regression, though more
complex models have also been proposed. The combinatorial nature of miRNA
action, however, makes expression-based strategies difficult, as most
transcripts are regulated by several miRNAs simultaneously, the contribution
of individual miRNAs may be small, and most miRNAs target a large number of
transcripts \citep{Baek2008}.

Cancer is a genetic disease caused by mutations in the genome of tumor cells \citep{Hanahan2000}.
Some of these mutations can be inherited, while some arise during the life-
time of an individual. To understand how cancer develops, it is of
paramount importance to identify genes which contribute to the tumorigenesis.
It is also essential to determine genes, and other factors, that influence
cancer aggressiveness as well as treatment sensitivity and resistance so that better
and more targeted treatments can be developed and applied.

Breast cancer is the most common of female cancers and causes remarkable
morbidity and mortality word-wide. Annually more than 1.5 million women
develop breast cancer \citep{Ferlay2015}. Thus, breast cancer constitutes a
major global health problem. Previous studies comparing the expression
profiles of normal breast tissue and breast cancer tissue have revealed that
expression signatures classify breast cancers into distinct subtypes, which
are associated with prognosis \citep{Sorlie2001}. Recent studies have
suggested that microRNAs can explain some of the heterogeneity and pathology
of breast cancer and show promise as prognostic markers
\citep{vanSchooneveld2015}.

The aim of this thesis was to apply a recently proposed Bayesian variable
selection method to microRNA target discovery in breast cancer. Variable
selection was applied in the context of Bayesian regression of protein, mRNA
and miRNA expression profiles, to elucidate which microRNAs are
relevant in regulating protein expression levels. The prediction results were
compared with lasso regression, a popular method for target prediction, and
with known validated microRNA targets.
