
\section{Introduction}
%% Leave first page empty
\thispagestyle{empty}

\textbf{REFET PUUTTUU!}

Breast cancer is the most common of female cancers and causes remarkable
morbidity and mortality word-wide. Annually more than 1.5 million women
develop breast cancer. Thus, breast cancer constitutes a major global health
problem.

Cancer is a genetic disease caused by mutations in the genome of the cancer
cells. Some of these mutations can be inherited, while some are somatic, that
is, mutations that arise during the life-time of an individual in the tissue,
where cancer develops. To understand how breast cancer develops,
it is of paramount importance to identify genes which
contribute to the tumorigenesis. It is also essential to identify
genes, and other factors, that influence cancer aggressivity and
treatment sesivity and resistance so that better and more targeted
treatments can be developed and applied in the clinic.
% It is already known that many of
% these mutations perturb the expression of the gene in question.

Previous studies comparing the expression profiles of normal breast tissue
and breast cancer tissue have revealed expression signatures
classify breast cancers into distinct subtypes that are associated with prognosis.

MicroRNAs (miRNAs) are short single-stranded RNA-molecules that act in
post-transcriptional regulation of gene expression. They have been found in a
wide variety of animals and plants, and also in viruses. MicroRNAs are higly
conserved in evolution and function in diverse developmental, physiological
and pathological processes. miRNAs have been indicated as potential
causative agents in numerous diseases, including several types of cancer.
Therefore, the study of microRNAs and their function in cancer can offer
insights into tumorigenesis and cancer progression as well as potential new
biomarkers and treatments.

A plethora of different methods for computational identification of microRNA
target genes have been proposed. Early methods were based on sequence
similarity of the microRNA and messenger RNA of putative target genes. More
recent methods use expression profiles to either augment sequence data or as
sole predictors for interaction. Expression methods are mostly based on
silmilarity measures, such as correlation, or variations of multiple
linear regression. The combinatorial nature of miRNA action, however, makes
expression based strategies difficult, as one transcript may be regulated by
several different miRNAs simultaneously and the contribution of each miRNA may
be small. Additionally, most miRNAs target a very large number of transcripts.

The aim of this thesis was to apply a recently proposed Bayesian variable
selection method to microRNA target discovery in breast cancer. The variable
selection was applied in the context of Bayesian regression of expression
profiles to elucidate which microRNAs are highly relevant in regulating
protein expression levels. The results were compared with a similar recent
study and also with known validated microRNA targets as well as microRNAs
previously indicated in breast cancer.
