
%%% This file contains the materials and methods section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Research material and methods}\label{material-and-methods}

\subsection{Research material}

The data analysed in this thesis consists of 283 tumor samples collected from
280 breast cancer patients treated in two Norwegian hospitals. Protein, mRNA
and microRNA expression were measured from each sample. The data were
published by \citet{Aure2015} and are publicly available.

The data are part of the Oslo2 cohort, which consists of breast cancer
patients with primarily operable disease -- that is stage cT1--cT2 -- treated
in several Norwegian hospitals. Collection of the cohort started in 2006 and
is still ongoing. Therefore, no survival data were available for analysis.
Clinical data for included patients are summarised in Table \ref{clinical-
data}. No control samples of healthy breast tissue were available.

\input{clinical_table}

The mRNA and microRNA expression were measured using Agilent Technologies
SurePrint G3 Human GE 8x60K and Human miRNA Microarray Kit (V2) microarrays,
respectively. These contain 27958 genes and 887 miRNAs according to
manufacturer annotation.

MicroRNAs detected in less than 10\% of samples had been removed from the
publicly available data, leaving 421 miRNAs. Out of these, eleven miRNAs (hsa-
miR-1274a, hsa-miR-1274b, hsa-miR-1280, hsa-miR-1308, hsa- miR-1826, hsa-
miR-1974, hsa-miR-1975, hsa-miR-1977, hsa-miR-1979, hsa-miR-720, hsa-
miR-886-3p) were reported as missing by miRBase Tracker. Reviewed on miRBase,
these miRNAs are reported as being fragments of other RNA species (e.g. tRNA
or rRNA) and, thus, removed from the database. Therefore, these eleven miRNAs
were removed from subsequent analyses.

Protein expression was measured using a reverse phospatase protein array
(RPPA) containing a set of 105 proteins, most of which are found on the PI3K--
pathway \textbf{(ref to G.Mills?)}. Only the gene expression values
corresponding to each measured protein were used in the analysis. Analyses
performed in this thesis used the publicly available preprocessed data
\textbf{(paitsi jos ehtii tehä vielä preprocin ite)}.





% The mRNA and microRNA expression data are publicly available in preprocessed
% format in the Gene Expression Omnibus (GEO) database \citep{GEO} under
% accession IDs xx and xx respectively. For the purpose of this thesis, the raw
% Agilent expression data were kindly provided and used for the analyses instead
% of the preprocessed data. The protein expression data is available in
% Additional file 4 of \citet{norjis} also in preprocessed format.

% Clinical data concerning each patient and cancer were also provided. A summary
% of the clinical parameters is presented in table \ref{clinical-data}. The
% predominant tumor type in the data was ductal carcinoma, which is in general
% the most common histological type of breast cancer.

% Use danish data for validation?


\subsection{Methods}

All computational analyses were performed in R \citep{R} and workflow
management was handled with Anduril \citep{OvaskaXXXX}. Monte-Carlo
simulations for the Bayesian regression models were done using RStan
\citep{stan} and all simulations were were performed using computer
resources within the Aalto University School of Science "Science-IT"
project.




\subsubsection{Preprocessing data}

\subsubsection{Variable selection with projection prediction}

The assumed number of relevant miRNAs, $p_n$, was estimated as follows.
Ensembl gene ID's were downloaded for all protein coding genes in the human
genome using biomaRt \citep{biomaRt}. A sample of 1000 genes was then taken
from these. Known validated microRNA interaction partners for each sampled
gene were downloaded from miRWalk. Genes for which there were no validated
miRNA interactors were assumed to have none. The mean number of miRNA
interactors per gene was the computed, which resulted in the estimate
$\hat{p_n} \approx 14$.
