%!TEX root = dippa.tex
%%% This file contains the materials and methods section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Research material and methods}\label{material-and-methods}

\subsection{Research material}

The data analysed in this thesis consists of 283 tumor samples collected from
280 breast cancer patients treated in two Norwegian hospitals. Protein, mRNA
and microRNA expression were measured from each sample. The data were
published by Aure et al. \citep{Aure2015} and are publicly available. Analyses
performed in this thesis used the publicly available preprocessed data.

The data are part of the Oslo2 cohort, which consists of breast cancer
patients with primarily operable disease -- that is stage cT1--cT2 -- treated
in several Norwegian hospitals. Collection of the cohort started in 2006 and
is still ongoing. Therefore, no survival data were available for analysis.
Clinical data for included patients were kindly provided by Aure and
associates and a compiled summary is presented in Table
\ref{clinical-data-numeric} and Table \ref{clinical-data-nominal}.
No control samples of healthy breast tissue were available.

\input{clinical_table}

The mRNA and microRNA expression were measured using Agilent Technologies
SurePrint G3 Human GE 8x60K and Human miRNA Microarray Kit (V2) microarrays,
respectively. These microarrays measure 27958 genes and 887 miRNAs, according to
manufacturer annotation.

Protein expression was measured using a reverse phospatase protein array
(RPPA) containing a set of 105 proteins, most of which are found on the PI3K--
pathway \textbf{(ref to G.Mills?)}. Only the gene expression values
corresponding to each measured protein were used in the analysis.





% The mRNA and microRNA expression data are publicly available in preprocessed
% format in the Gene Expression Omnibus (GEO) database \citep{GEO} under
% accession IDs xx and xx respectively. For the purpose of this thesis, the raw
% Agilent expression data were kindly provided and used for the analyses instead
% of the preprocessed data. The protein expression data is available in
% Additional file 4 of \citet{norjis} also in preprocessed format.

% Clinical data concerning each patient and cancer were also provided. A summary
% of the clinical parameters is presented in table \ref{clinical-data}. The
% predominant tumor type in the data was ductal carcinoma, which is in general
% the most common histological type of breast cancer.

% Use danish data for validation?


\subsection{Methods}

All computational analyses were performed in R \citep{R} and workflow
management was handled with Anduril \citep{OvaskaXXXX}. Monte-Carlo
simulations for the Bayesian regression models were done using RStan
\citep{stan} and all simulations were were performed using computer
resources within the Aalto University School of Science "Science-IT"
project. Publicly available data were downloaded from the
Gene Expression Omnibus (GEO) using the R package GEOquery.




\subsubsection{Preprocessing and quality control}

Publicly available expression data, which are in preprocessed form, were used
for all analyses. No further preprocessing was done to ensure that any
differences in results were due to actual analyses and not caused by the
preprocessing step. For details on the preprocessing, the reader is referred
to the supplementary data of Aure et al \citep{Aure2015}.

MicroRNAs detected in less than 10\% of samples had been removed from the
public data, leaving 421 miRNAs. Out of these, eleven miRNAs (hsa-
miR-1274a, hsa-miR-1274b, hsa-miR-1280, hsa-miR-1308, hsa- miR-1826, hsa-
miR-1974, hsa-miR-1975, hsa-miR-1977, hsa-miR-1979, hsa-miR-720, hsa-
miR-886-3p) were reported as missing by miRBase Tracker. Reviewed on miRBase,
these miRNAs are reported as being fragments of other RNA species (e.g. tRNA
or rRNA) and, thus, removed from the database. Therefore, these eleven miRNAs
were removed from subsequent analyses, leaving 410 miRNAs.

Explain QC plots:
\begin{itemize}
	\item hierarchical clustering of samples
	\item distributions of chips
\end{itemize}

Correlation analyses.


\subsubsection{Variable selection with projection prediction}

For predicting protein expression from mRNA and miRNA
expression, a similar regression model to Aure et al
was used:
\begin{equation}
	\mathbf{y_j} = w_0 + w_g \dot \mathbf{x_j} + \mathbf{w_j} \cdot \mathbf{Z} + \mathbf{\epsilon_j},
\end{equation}
where $y_j$ denotes the protein expression vector for the protein $j$ which
is produced from translation of the mRNA $j$ ($j = (1, ..., K)$), $w_g$
is the regression coefficient for mRNA $j$, and $w_0$ is the intercept term
(for a derivation of this equation, see Aure et al \citep{Aure2015}).
The intercept and mRNA coefficients were given Gaussian prior
distributions with zero mean and standard deviation $\sigma=5$.
A hierarchical shrinkage prior was applied to the miRNA coefficients
$\mathbf{w_j}$.

Explain:
\begin{itemize}
  \item hierarchical shrinkage priors
  \item projection prediction variable selection
  \item justification for variance of tau prior ($p_n$ van Der Pass)
\end{itemize}

The assumed number of relevant miRNAs, $p_n$, was estimated as follows.
Ensembl gene ID's were downloaded for all protein-coding genes in the human
genome using biomaRt \citep{biomaRt}. From these, a sample of 1000 genes was
taken, and known validated microRNA interaction partners for each sampled gene
were downloaded from miRWalk \citep{Dweep2015}. Genes for which there were no
validated miRNA interactors were assumed to have zero. The mean number of
miRNA interactors per gene was the computed, which resulted in the estimate
$\hat{p_n} = 13.75$.

The coefficient of determination, $R^2$ \textbf{(kaava!)}, was used as a measure of final model
fit. $R^2$ is a favorable measure of goodness of fit, because it is invariant
to variable scaling. As expression data do not have a well defined scale,
$R^2$ is well-suited in this sense. One problem with $R^2$ is that it
increases with the addition of more explanatory variables. The adjusted $R^2$,
defined as $\bar{R}^2 = 1-(1-R^2)*(n-1)/(n-p)$ (where $n$ is the number of
observations and $p$ the number of explanatory variables) adjusts for the
number of regressors relative to the number of observations, thus penalizing
for including additional variables. $\bar{R}^2$ was used for comparing the
final model with a model including only the gene expression variable.

Possible extras:
\begin{itemize}
	\item DE miRNA analysis between triple negatives and others
	\item bipartite network from found MTIs and some network analysis
\end{itemize}
