%!TEX root = dippa.tex
%%% This file contains the materials and methods section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Materials and methods}\label{materials-and-methods}

\subsection{Research material}

The data analysed in this thesis consists of 283 tumor samples collected from
280 breast cancer patients treated in two Norwegian hospitals. Protein, mRNA
and microRNA expression were measured from each sample. The data were
published by Aure et al. \citep{Aure2015} and are publicly available. Analyses
performed in this thesis used the publicly available preprocessed data.

The data are part of the Oslo2 cohort, which consists of breast cancer
patients with primarily operable disease -- that is stage cT1--cT2 -- treated
in several Norwegian hospitals. Collection of the cohort started in 2006 and
is still ongoing. Therefore, no survival data were available for analysis.
Clinical data for included patients were kindly provided by Aure and
associates and a compiled summary is presented in Table
\ref{clinical-data}. Notably, the vast majority of tumors in the data
were ductal carcinomas, which is in general the most common histological type
of breast cancer. Patient ages ranged from X to Y with a median of 45 years.
No matched control samples of healthy breast tissue were available.

\input{clinical_table}

The mRNA and microRNA expression were measured using Agilent Technologies
SurePrint G3 Human GE 8x60K and Human miRNA Microarray Kit (V2) microarrays,
respectively. These microarrays measure 27958 genes and 887 miRNAs, according
to manufacturer annotation. Protein expression was measured using a reverse
phospatase protein array (RPPA) containing a set of 105 proteins. Most of the
proteins are found on the PI3K/AKT intracellular pathway, which is important
for cell-cycle regulation and, thus, cancer. \textbf{(ref to G.Mills?)}.

% The mRNA and microRNA expression data are publicly available in preprocessed
% format in the Gene Expression Omnibus (GEO) database \citep{GEO} under
% accession IDs xx and xx respectively. For the purpose of this thesis, the raw
% Agilent expression data were kindly provided and used for the analyses instead
% of the preprocessed data. The protein expression data is available in
% Additional file 4 of \citet{norjis} also in preprocessed format.

% Clinical data concerning each patient and cancer were also provided. A summary
% of the clinical parameters is presented in table \ref{clinical-data}. The
% predominant tumor type in the data was ductal carcinoma, which is in general
% the most common histological type of breast cancer.

% Use danish data for validation?





\subsection{Methods}

All computational analyses were performed in R and workflow management was
handled with Anduril \citep{Ovaska2010}. Monte-Carlo simulations for the
Bayesian regression models were performed with RStan
\citep{RStan} using the No-U-Turn variant of a Hamiltonian Monte Carlo
algorithm for sampling posterior distributions. All simulations were were run using
computer resources within the Aalto University School of Science "Science-IT"
project.



\subsubsection{Preprocessing and quality control}

MicroRNA and mRNA expression data were downloaded from the Gene Expression
Omnibus (accessions GSEXXX and GSEXXX, respectively \citep{Edgar2002}) using
the GEOquery R package \citep{GEOquery}. Protein expression data were
downloaded as a Microsoft Excel sheet from the supplementary data of Aure et
al \citep{Aure2015}. Data were cleaned using R. Notably, the protein
data have only a single set of measurements for two sets of related genes,
namely AKT1, AKT2, AKT3 and (labeled respectively as "AKT1/2/3" and "" in the data).
The same values from the single measurement set were used for each
of the related genes. Presumably the protein array used cannot differentiate
the proteins of these genes, no mention of this is made by Aure et al.
Separate measurements for all genes are included in the mRNA expression data.

Out of the 421 microRNAs present in the public dataset, eleven miRNAs (hsa-
miR-1274a, hsa-miR-1274b, hsa-miR-1280, hsa-miR-1308, hsa- miR-1826, hsa-
miR-1974, hsa-miR-1975, hsa-miR-1977, hsa-miR-1979, hsa-miR-720, hsa-
miR-886-3p) were reported as missing from miRBase by miRBase Tracker. Reviewed
on miRBase, these miRNAs are reported as being fragments of other RNA species
(e.g. tRNA or rRNA) and, thus, removed from the database. The eleven miRNAs
were therefore removed from subsequent analyses, leaving 410 miRNAs.

The publicly available expression data are in preprocessed form, and as such,
no further preprocessing of the actual measurements was done. For details on
the preprocessing, the reader is referred to the supplementary data of Aure et
al \citep{Aure2015}. For regression analyses, all variables (miRNA, mRNA and
protein) were scaled to have zero mean and unit variance, a commonly
used transformation in regression.

For assessing the quality of the data, distributions of each microarray and each
variable (miRNA, mRNA and protein) were plotted. These plots are presented in
Appendix \ref{app:qc-plots}. Two observations from these plots are worth noting here.
Several genes have significant outliers in the protein data. The miRNA
microarrays have distributions that are highly skewed towards very small
values. This is indicative of the generally low abundances of miRNA molecules
and, although partly due to different preprocessing compared to the mRNA and
protein arrays, it raises suspicion of significant noise in the miRNA data.
The raw chip data produced from microarray readers could not be assessed, as
these were not available.

The data have been collected from two different hospitals and to assess
possible bias introduced by the separate sites, a principal component
analysis (PCA) and hierarchical clustering of samples were performed
separately for each data type. The first two principal components
for each data type were plotted and are shown in Figure \ref{fig:pca}.
No significant batch effect is visible. The hierarchical clustering (not shown)
gave a similar result.

Correlation analyses.


\subsubsection{Variable selection with projection prediction}\label{sec:PPVS}

For predicting protein expression from mRNA and miRNA expression, a similar
regression model to Aure et al was used (here referred to as the \emph{full
model}):
\begin{equation}
	y_j = w_0 + z_j w_g + X w_j + \epsilon_j,
\end{equation}
where $y_j$ denotes the protein expression vector for the protein $j$ which
is produced from translation of the mRNA $j$ ($j = (1, ..., K)$), $w_g$
is the regression coefficient for mRNA $j$, and $w_0$ is the intercept term
(for a justification of this equation, see Aure et al \citep{Aure2015}).
A model with only the mRNA expression covariate (called the \emph{gene-only} model),
defined as $y = w_0 + z w_g + \epsilon$, was used as a baseline for the full model.
A normally ditributed error term with equal errors and no correlation
between observations was assumed for all models.

For Bayesian regression, the likelihood was therefore defined as
\begin{equation}
	y | w_g, w, \sigma, z, X \sim N(Xw, \sigma^2I),
\end{equation}
The intercept and mRNA coefficients were given Gaussian prior
distributions with zero mean and standard deviation $\sigma=5$.
A hierarchical shrinkage prior was applied to the miRNA coefficients
$w_j$.

Explain:
\begin{itemize}
  \item hierarchical shrinkage priors
  \item projection prediction variable selection
  \item justification for variance of tau prior ($p_n$ van Der Pass)
\end{itemize}

The assumed number of relevant miRNAs, $p_n$, was estimated as follows.
Ensembl gene ID's were downloaded for all protein-coding genes in the human
genome using biomaRt \citep{biomaRt}. From these, a sample of 1000 genes was
taken, and known validated microRNA interaction partners for each sampled gene
were downloaded from miRWalk \citep{Dweep2015}. Genes for which there were no
validated miRNA interactors were assumed to have zero. The mean number of
miRNA interactors per gene was used an estimate, giving
$\hat{p_n} = 13.75$.

The coefficient of determination, $R^2$ \textbf{(kaava!)}, was used as a measure of final model
fit. $R^2$ is a favorable measure of goodness of fit, because it is invariant
to variable scaling. As expression data do not have a well defined scale,
$R^2$ is well-suited in this sense. One problem with $R^2$ is that it
increases with the addition of more explanatory variables. The adjusted $R^2$,
defined as $\bar{R}^2 = 1-(1-R^2)*(n-1)/(n-p)$ (where $n$ is the number of
observations and $p$ the number of explanatory variables) adjusts for the
number of regressors relative to the number of observations, thus penalizing
for including additional variables. $\bar{R}^2$ was used for comparing the
full model with the gene only model.

Possible extras:
\begin{itemize}
	\item DE miRNA analysis between triple negatives and others
	\item bipartite network from found MTIs and some network analysis
\end{itemize}
