%!TEX root = dippa.tex
%%% This file contains the materials and methods section of my master's thesis.
%%% Author: Viljami Aittomäki


\section{Materials and methods}\label{materials-and-methods}

\subsection{Research material}

The data analyzed in this thesis consists of 283 tumor samples collected from
280 breast cancer patients treated in two Norwegian hospitals. Protein, mRNA
and microRNA expression were measured from each sample. The data were
published by Aure et al \citep{Aure2015} and are publicly available.

The data are part of the larger Oslo2 cohort, which consists of breast cancer
patients with primarily operable disease, that is stage cT1--cT2. Collection
of the cohort started in 2006 and is still ongoing. Therefore, no survival
data were available for analysis. 
% Clinical data for included patients were
% kindly provided by Aure and associates and a compiled summary is presented in
% Table \ref{clinical-data}.
Patient ages ranged from 30 to 83 years with a median of 55 years. The vast
majority of tumors in the data were ductal carcinomas, which is the most
common type of breast cancer, however, in general the dataset consisted of a
heterogeneous collection of different histological types and stages of breast
cancer tumors. No matched control samples of healthy breast tissue were
available.

%\input{clinical_table}

The mRNA and microRNA expression were measured using Agilent Technologies (Santa Clara, CA, USA)
SurePrint G3 Human GE 8x60K and Human miRNA Microarray Kit (V2) microarrays,
respectively. These microarrays measure 27958 genes and 887 miRNAs, according
to manufacturer annotation. Protein expression was measured with reverse-phase
protein arrays (RPPA) \citep{Tibes2006} for a selected set of 105 proteins relevant in
cancer. Most of the proteins are found on the PI3K/AKT intracellular pathway,
which is important for cell-cycle regulation.




\subsection{Methods}

A Bayesian regression model of protein, mRNA and miRNA expression data was
constructed, and projection predictive variable selection was used to predict
microRNA targets in breast cancer data. Details of the methods used are
presented in this section.

All computational analyses were performed in R
\citep{R} and workflow management was handled with Anduril \citep{Ovaska2010}.
Monte-Carlo simulations for the Bayesian regression models were computed with
\emph{RStan} \citep{RStan} using the No-U-Turn variant of a Hamiltonian Monte Carlo
algorithm for sampling posterior distributions. The simulations were performed using
computer resources within the Aalto University School of Science "Science-IT"
project.


\subsubsection{Preprocessing and quality control}

MicroRNA and mRNA expression data were downloaded from the Gene Expression
Omnibus \citep{Edgar2002} (accessions GSE8210 and GSE8212, respectively) using
the GEOquery R package \citep{Davis2007}. Protein expression data were
downloaded as a Microsoft Excel sheet from the supplementary data of Aure et
al \citep{Aure2015}.

Notably, the protein data have only a single set of measurements for two sets
of related genes, namely AKT1, AKT2, AKT3 and GSK3A, GSK3B (labeled
respectively as "AKT1/2/3" and "GSK3A/GSK3B" in the original data). Presumably
the protein array used cannot differentiate these proteins, but
no mention of this is made by Aure et al. For the analyses presented here,
each of these genes was considered separately, and the same expression values
from the single measurement set were used for all of the related genes
respectively.

Out of the 421 microRNAs present in the public dataset, eleven miRNAs
(hsa-miR-1274a, hsa-miR-1274b, hsa-miR-1280, hsa-miR-1308, hsa- miR-1826,
hsa-miR-1974, hsa-miR-1975, hsa-miR-1977, hsa-miR-1979, hsa-miR-720,
hsa-miR-886-3p) were reported as missing from miRBase by miRBase Tracker \citep{VanPeer2014}.
Reviewed on miRBase, these miRNAs are reported as being fragments of other RNA species
(e.g. tRNA or rRNA) and, thus, removed from the database. The eleven miRNAs
were therefore removed from subsequent analyses, leaving 410 miRNAs.

The publicly available expression data are in preprocessed form and 
in log2 scale (as per convention for expression data). As such,
no further preprocessing of the actual measurements was done. For details on
the preprocessing, the reader is referred to the supplementary data of Aure et
al \citep{Aure2015}. The mRNA expression data are available as probe-level
measurements, these were summarized to gene-level using manufacturer probe
annotations by taking the mean of all probes targeting the same gene.
For regression analyses, all variables (miRNA, mRNA
and protein) were further scaled to have zero mean and unit variance,
a commonly used transformation.

For assessing the quality of the expression data, distributions of each
microarray and each variable (miRNA, mRNA and protein) were plotted. The data
have been collected from two different hospitals and to assess possible bias
introduced by the separate sites, a principal component analysis (PCA) and
hierarchical clustering of samples were performed separately for each data
type.



\subsubsection{Validated target reference}

The predicted miRNA targets from both projection prediction and lasso
regression were compared to validated targets in DIANA-TarBase v7.0
\citep{Vlachos2015} (referred to as TarBase from here on) and miRTarBase
release 6.0 \citep{Chou2016}. Data from both databases were downloaded and a
union of the databases was used as a reference set of validated interactions.
The resulting set contained 328825 validated miRNA-mRNA interactions, out of
which only 4082 were between a gene and miRNA present in the analyzed data,
however.




\subsubsection{Correlation analysis}

To assess dependencies between variables from the different expression data
types (protein, mRNA, miRNA) and the validity of correlation as a target
prediciton tool, Pearson's correlations were computed between matched and
unmatched protein-mRNA pairs (where matched refers to both corresponding to
the same gene) as well as validated and random protein-miRNA and mRNA-miRNA
pairs, where validated pairs were ones present in the reference set described
above. The random correlations consisted of 5000 randomly picked pairs (with
replacement).




\subsubsection{Regression models}

For predicting protein expression from mRNA and miRNA expression, a similar
regression model to Aure et al was used:
\begin{equation}
	\label{eq:reg-model}
	y = \beta_0 + z \beta_g + X \beta + \epsilon,
\end{equation}
where $y$ is the protein expression and $z$ the
mRNA expression for gene $k$ ($k$ is suppressed for clarity), $w_g$
is the regression coefficient for the mRNA, $X$ is the matrix of miRNA
expression vectors, and $w_0$ is the intercept term
(for a justification of this equation, see \citep{Aure2015}).
A separate model was fitted for each gene.
A model with only the mRNA expression covariate (called the \emph{gene-only model}),
defined as $y = \beta_0 + z \beta_g + \epsilon$, was used as a baseline.
A normally distributed error term with equal errors and no correlation
between observations was assumed for all models.

The likelihood for Bayesian regression was therefore defined as
\begin{equation}
	y | \mathbf{\beta}, \sigma, z, X \sim N(\beta_0 + z \beta_g + X \beta, \sigma^2I),
\end{equation}
where $\mathbf{\beta} = [\beta_0, \beta_g, \beta]$ for convenience.
The intercept and mRNA coefficient were given diffuse Gaussian priors
and $\sigma$ a uniform prior:
\begin{subequations}
  \begin{align}
    \beta_0 & \sim N(0, 5^2) \\
    \beta_g & \sim N(0, 5^2) \\
    \sigma  & \propto 1.
  \end{align}
\end{subequations}
A hierarchical shrinkage prior was applied to the miRNA coefficients $\beta$,
as defined in Equation \eqref{eq:hs-prior}. The degrees of freedom for the
$\lambda_j$ priors was set at $\nu=3$ (similar to Piironen and Vehtari
\citep{Piironen2015}). The prior for $\tau$ was defined as:
\begin{equation}
    \tau \sim \textup{half-Cauchy} \left ( 0, \frac{p_n}{n}\sqrt{\textup{log}(n/p_n)} \right ),
\end{equation}
combining the previous suggestions of half-Cauchy and fixed $tau$.
The assumed number of relevant miRNAs, $p_n$, was estimated as follows.
Ensembl gene ID's were downloaded for all protein-coding genes in the human
genome using biomaRt \citep{Durinck2009}. From these, a sample of 1000 genes was
taken, and known validated microRNA interaction partners for each sampled gene
were downloaded from miRWalk \citep{Dweep2015}. Genes for which there were no
validated miRNA interactors were assumed to have zero. The mean number of
miRNA interactors per gene was used as an estimate, giving
$\hat{p_n} = 13.75$.\footnote{Note that this is probably an underestimate of
the true $p_n$, as the set of experimentally validated miRNA targets is
very likely to underrepresent the true \emph{in vivo} set of miRNA targets.}



\subsubsection{Variable selection}\label{sec:methods-variable-selection}

Projection predictive variable selection (as described in Section
\ref{sec:bayes-variable-selection} and as applied by Piironen and Vehtari \citep{Piironen2015}),
was used to obtain the relevant set of microRNAs for each gene. A full
reference model was fitted by drawing 2000 samples from the posterior using
RStan (with 4 chains, 1000 samples each and the first half discarded as burn-in).
A random sample of $S=1000$ simulation samples from the full posterior was used
to increase projection speed.

A series of projected submodels was obtained using a
forward search strategy, that is, the search started from a model including
only the intercept, the mRNA expression $z$ was always added as the first
covariate, and at each subsequent step, the miRNA covariate $x_j$ giving the
largest decrease in discrepancy between the reference and projected models 
(Eq. \eqref{eq:model-discrepancy}) was
chosen. The forward search was continued up to 200 variables.

For choosing the model size, 10-fold cross validation (CV) was used, as proposed by
Piironen and Vehtari \citep{Piironen2016}. This means that the above model selection
process was repeated $K=10$ times, each time leaving out $n/K$ observations for
model evaluation. For judging the appropriate model size, estimation of model
predictive performance was performed as explained below.

\paragraph{Model predictive performance}
The predictive performance of each submodel was evaluated using 
a log predictive density.
Given submodel $M_\perp$ with the posterior predictive distribution
$p(\tilde{y}| \tilde{z}, \tilde{X}_\perp, \mathbf{\beta}_\perp, \sigma_\perp, D_\perp)$,
where $D_\perp$ is the observed data in the current submodel and CV fold and
$(\mathbf{\beta}_\perp, \sigma_\perp)$ the projected model parameters, the logarithm of the predictive density
(LPD) was computed at each of the left-out observations $(y_*, z_*, X_*)$.
The LPD was estimated by averaging over the $S$ simulated posterior samples:
\[
	\textup{LPD}_*(M_\perp) \approx \textup{log}\frac{1}{S} \sum_s^S p(y_*| z_*, X_*, \mathbf{\beta}_\perp, \sigma_\perp, D_\perp).
\]
The LPD's from each CV fold were pooled and a mean over the full set of data
(MLPD) was used as a summary. To compare the predictive performance of a
submodel to the full reference model, the difference in MLPD ($\Delta \textup{MLPD}$) was
computed. Bayesian bootstrap \citep{Rubin1981} (with 5000 samples) was used to estimate a
distribution for $\Delta \textup{MLPD}$ as:
\[
	\Delta \textup{MLPD}^{b}(M_\perp,M_{\textup{r}}) = \sum_i^n w_i^{(b)} \left [ \textup{LPD}_i(M_\perp) - \textup{LPD}_i(M_{\textup{r}}) \right ],
\]
where $w_i^{(b)}, i = 1, \dotsc,n$, are the bootstrap weights
for the $b$'th bootstrap sample (subject to $\sum_i w_i^{(b)} = 1$).
The \emph{bayesboot} R package was used for computing the bootstrap.
The $\Delta \textup{MLPD}(M_\perp,M_{\textup{r}})$ was then used
as an estimate of the predictive performance of submodel $M_\perp$.

\paragraph{Choosing model size}
For choosing the model size, the following condition was used:
\begin{equation}
	\label{eq:size-condition}
	\textup{Pr}(\Delta \textup{MLPD}(M_\perp,M_{\textup{r}}) > U) \geq \alpha,
\end{equation}
where $U = \gamma \Delta \textup{MLPD}(M_0,M_{\textup{r}})$, and $M_0$
refers to the intercept-only model. The number of covariates in the smallest model satisfying this condition
was chosen as the final model size. This means that model size was chosen
such, that the probability of the projected model improving performance
by a constant factor ($\gamma$) over the intercept-only model
was at least $\alpha$. The choice of values for $\gamma$ and $\alpha$
reflects accepted loss in predictive preformance (and related
uncertainty) compared to the reference model; several values for
both were tested.

\paragraph{Final model selection}
The final projected model was obtained by reapplying the projection search up to
the chosen number of variables, using all data for each gene. miRNAs included
in each gene's model were considered putative target interactors. miRNAs for
which the 95\% posterior interval did not include the origo were considered
\emph{significant}. For some genes the condition in \eqref{eq:size-condition} was not
met after including 200 covariates. In these cases it was concluded that the
miRNA covariates provided no additional information on the protein expression
and, thus, none of them were deemed as targeting the gene. In some cases the
condition was met already by the model with only the mRNA covariate, and the
same conclusion was made.

\paragraph{Lasso regression}
A lasso regression model was also fitted for each gene using the
\emph{glmnet} R package. In this case the mRNA variable was treated equal
to the miRNA variables and subjected to the lasso regularization. For choosing
the regularization parameter $\lambda$, the default algorithm of \emph{glmnet}
was used. That is, models were fit for a decreasing sequence of $\lambda$
values, and 10-fold cross validation was used to compute a mean square
prediction error (MSE) at each $\lambda$. The largest value of $\lambda$ (i.e.
sparsest model) that had a MSE at most one standard deviation apart from the
lowest MSE attained in the cross validation was used. The covariates included
in the model using the chosen $\lambda$ were considered putative target
interactors. For some genes, this criterion was met by the intercept-only
model, and again, in these cases none of the miRNAs were deemed as targeting
the gene.



\subsubsection{Measuring model fit}

To assess convergence of simulations, the potential scale reduction
measure $\hat{R}$ proposed by Gelman et al was used \citep{Gelman2013}.
The coefficient of determination $R^2$ was used as a measure
of regression model performance. $R^2$ is defined as
\begin{equation}
	R^2 = \frac{SS_{\textup{residuals}}}{SS_{\textup{total}}} = \frac{\sum_{i=1}^{n}(\hat{y}_i-y_i)^2}{\sum_{i=1}^{n}(y_i-\bar{y})^2},
\end{equation}
where $\hat{y}_i$ are the predictions made by the model and $\bar{y}$ is the
mean of the outcome variable $y$. $R^2$ corresponds to the proportion of variance
of the outcome variable that is explained by a statistical model, and can be
used as a measure of model fit. $R^2$ has the property of being invariant to
variable scaling, which makes it suitable for use with expression data, as
expression data do not have a well defined scale.

An important caveat of $R^2$ is that in linear regression it often increases
monotonically by adding more explanatory variables. The adjusted $R^2$,
defined as $\bar{R}^2 = 1-(1-R^2)\frac{n-1}{n-p}$ (where $n$ is the number of
observations and $p$ the number of explanatory variables) adjusts for the
number of regressors relative to the number of observations, thus penalizing
inclusion of additional variables. Therefore, $\bar{R}^2$ was used for comparing the
projected model with the gene only model.
